<!DOCTYPE html>
<html lang="en"><!-- InstanceBegin template="/Templates/subpage.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<!-- InstanceBeginEditable name="meta" -->
<!-- InstanceEndEditable -->
<!--[if gte IE 9]><meta http-equiv="X-UA-Compatible" content="IE=Edge"><![endif]-->
<!-- InstanceBeginEditable name="doctitle" -->
<title>Chapman University</title>
<!-- InstanceEndEditable -->
<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" type="text/css" href="../css/harv.css">
<link rel="stylesheet" type="text/css" href="../css/harvdrop.css">
<!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><script src="/js/html5shiv.js"></script><![endif]-->
<script src='http://code.jquery.com/jquery-latest.min.js' type='text/javascript'></script><script src="/js/jquery.ui.totop.js"></script>
<!-- InstanceBeginEditable name="head" --><style>
.text_box, p {
	text-align: left !important;
}
#middle #mainsub .biblio-abstract-link.toggle {
}
</style>
<!-- InstanceEndEditable -->
</head>
<body>
<div id="container">
  <div id="smNavBk"><a href="http://compbiosciences.chapman.edu/index.php"><img src="../images/header400light.png" width="332" height="58" alt="Chapman University" style="float: left; margin-left: 10px; margin-top: 10px;"></a>
    <div id="collegeLinks" style="font-size: 24px;">
      <p><a href="https://www.chapman.edu/scst/"><strong><img src="../images/xdz1wyqhwtmxxekidsyy.jpg" width="35" height="35" alt=""/>Schmid College of Science and Technology</strong></a></p>
      <p><a href="http://www.chapman.edu/"><img src="../images/open-graph-cu-logo.png" alt="Chapman University" width="35" height="30" title="Chapman University"/> <span style="font-style: normal; font-size: 18pt;">Chapman University</span></a></p>
    </div>
    <div class="clear"></div>
  </div>
  </header>
  <nav id="topMenu" class="desktop"><!-- #BeginLibraryItem "/Library/topMenu.lbi" --> <nav id="topMenu" class="desktop"><ul id="TopNav" class="MenuBarHorizontal">
      <li> <a href="../index.php">Home</a></li>
      <li><a href="../members.php">Members</a>          
  <li><a href="../Research/research.php" class="MenuBarItemSubmenu">Research</a>
        <ul>
          <li><a href="../Research/computationalCancerBiology.php" >Computational Biology and Translational Bioinformatics in Cancer Research</a></li>
          <li><a href="../Research/proteinKinasesResearch.php" >Protein Kinases</a></li>
          <li><a href="../Research/molecularChaperonesAndCancerResearch.php">Computatoinal Studies of Molecular Chaperones</a></li>
          <li><a href="../Research/translationalBioinformatics.php" >Translational Bioinformatics and Drug Discovery</a></li>
          <li><a href="../Research/relatedResearch.php">Related Links</a></li>
        </ul>
  </li>
      <li><a href="publications.php" class="MenuBarItemSubmenu">Publications</a>          
        <ul>
          <li><a href="2015o2019.php">2015-2019</a></li>
          <li><a href="2010o2014.php">2010-2014</a></li>
          <li><a href="2005o2009.php">2005-2009</a></li>
          <li><a href="2000o2004.php">2000-2004</a></li>
          <li><a href="1995o1999.php">1995-1999</a></li>
          <li><a href="1990o1994.php">1990-1994</a></li>
          <li><a href="o1990.php">Before 1990</a></li>
        </ul>
    <li><a href="../aboutUs.php">About Us</a>  </li>
    <li><a href="../contact.php">Contact Us</a></li>
    </ul>
    <div class="clear"></div></nav><!-- #EndLibraryItem --></nav></div>
  <div id="middle">
    <div id="mainsub"><!-- InstanceBeginEditable name="crumbs" --><script>breadcrumb()</script><!-- InstanceEndEditable -->
     
     
     
    <div class="clear"></div>
    
    
   <!-- InstanceBeginEditable name="main_content_center" -->
   <p class="header">Publications: 2000-2004</p>
 
   <p class="sub_ctrXS">2004</p>

    G. M. Verkhivker.
    <a href="http://www.sciencedirect.com/science/article/pii/S1093326303001906"> Computational analysis of ligand binding dynamics at the intermolecular hot spots with the aid of simulated tempering and binding free energy calculations. </a>
    J. Mol.  Graph. Model. 22:335-348, 2004.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=15099830"> PUBMED. </a>
    <a href="#abstractComp" id="a1">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComp" id="b1"> 
      ABSTRACT: Equilibrium binding dynamics is studied for a panel of benzimidazole-containing compounds at the remodeled interface between human growth hormone (hGH) and the extracellular domain of its receptor (hGHbp), engineered by mutating to glycine hot spot residues T175 from the hormone and W104 from the receptor. Binding energetics is predicted in a good agreement with the experimental data for a panel of these small molecules that complement the engineered defect and restore the binding affinity of the wild-type hGH–hGHbp complex. The results of simulated tempering ligand dynamics at the protein–protein interface reveals a diversity of ligand binding modes that is consistent with the structural orientation of the benzimidazole ring which closely mimics the position of the mutated W104 hot spot residue in the wild-type hGH–hGHbp complex. This structural positioning of the benzimidazole core motif is shown to be a critical feature of the low-energy ligand conformations binding in the engineered cavity. The binding free energy analysis provides a plausible rationale behind the experimental dissociation constants and suggests a key role of ligand–protein van der Waals interactions in restoring binding affinity.
      <br><br></a_abstract>

    G. M. Verkhivker.
    <a href= "http://onlinelibrary.wiley.com/doi/10.1002/bip.20149/abstract">Protein conformational transitions  coupled to binding in molecular recognition of unstructured proteins: hierarchy of structural loss from all-atom Monte Carlo simulations of p27 unfolding–unbinding and structural  determinants of the binding mechanism.</a>
    Biopolymers 75:420-433, 2004.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=15468065"> PUBMED. </a>
    <a href="#abstractPro" id="a2">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractPro" id="b2"> 
      ABSTRACT: Conformational transitions coupled to binding are studied for the p27Kip1 protein which undergoes a functional disorder–to–order folding transition during tertiary complex formation with the phosphorylated cyclin A–cyclin-dependent kinase 2 (Cdk2) binary complex. Temperature–induced Monte Carlo simulations of p27Kip1 unfolding–unbinding carried out from the crystal structure of the tertiary complex have revealed a systematic trend in the hierarchy of structural loss for p27Kip1 and a considerable difference in mobility of p27Kip1 secondary structure elements. The most persistent interactions of p27Kip1 at the intermolecular interface during unfolding–unbinding simulations are formed by β-hairpin and β–strand that on average maintain their structural integrity considerably longer than other p27Kip1 elements. We have found that the ensemble of unfolded p27Kip1 conformations is characterized by transitions between mostly unbound, collapsed conformations and entropically favorable p27Kip1 conformations, which are weakly bound to the cyclin A side of the binary complex. The results of this study are consistent with the experimental evidence pointing to this region of the intermolecular interface as a potential initiation docking site during binding reaction and may reconcile conflicting experimental hypotheses on the recognition of substrate recruitment motifs.
      <br><br></a_abstract>

    G. M. Verkhivker.
    <a href= "http://link.springer.com/chapter/10.1007%2F0-306-48662-8_12"> A microscopic study of disorder-order transitions in molecular recognition of unstructured proteins: hierarchy of structural loss and the transition state determination from Monte Carlo simulations of p27Kip1 protein coupled unfolding and unbinding.</a>
    n Proceedings of 8th International School of Biophysics.  “Supramolecular structure and Function.”  p.199-230, 2004.
    <a href="#abstractMicro" id="a3">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractMicro" id="b3"> 
      ABSTRACT: It has been recently realized that a significant amount of protein domains and even entire proteins can lack intrinsic globular structure under physiological conditions, suggesting a reappraisal of the conventional structure-function paradigm. These proteins, termed as intrinsically unstructured intrinsically disordered or natively unfolded can largely comprise of disordered segments in their functional state. The intrinsic plasticity and functional disorder-order folding transitions coupled to binding can provide for these protein systems an important prerequisitefor effective molecular recognition, including high specificity coupled with low affinity, the ability to bind with several different targets, a precise control and simple regulation of the binding thermodynamics, and the increased rates of specific macromolecular association.
      <br><br></a_abstract>

    G. M. Verkhivker.
    <a href= "http://link.springer.com/chapter/10.1007%2F0-306-48662-8_13"> Computational detection of the binding site hot spots and predicting energetics of ligand binding at the remodeled  human growth hormone-receptor interface using a hierarchy of molecular docking and binding free energy approaches.</a>
    n Proceedings of 8th International School of Biophysics.  “Supramolecular structure and Function.”  p. 231-245, 2004.
    <a href="#abstractComp2" id="a4">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComp2" id="b4"> 
      ABSTRACT: 
      <br><br></a_abstract>

     
     
    <p class="sub_ctrXS">2003</p>
    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S.T. Freer, P. W. Rose.
    <a href= "http://onlinelibrary.wiley.com/doi/10.1002/prot.10456/abstract">Computational detection of the binding site hot spot at the remodeled human growth hormone-receptor interface.</a>
    Proteins: Struct. Funct. Genet.  53:203-219, 2003.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=14517972"> PUBMED. </a>
    <a href="#abstractComp3" id="a5">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComp3" id="b5"> 
      ABSTRACT: A hierarchical computational approach is used to identify the engineered binding-site cavity at the remodeled intermolecular interface between the mutants of human growth hormone (hGH) and the extracellular domain of its receptor (hGHbp). Multiple docking simulations are conducted with the remodeled hGH–hGHbp complex for a panel of potent benzimidazole-containing inhibitors that can restore the binding affinity of the wild-type complex, and for a set of known nonactive small molecules that contain different heterocyclic motifs. Structural clustering of ligand-bound conformations and binding free-energy calculations, using the AMBER force field and a continuum solvation model, can rapidly locate and screen numerous ligand-binding modes on the protein surface and detect the binding-site hot spot at the intermolecular interface. Structural orientation of the benzimidazole motif in the binding-site cavity closely mimics the position of the hot spot residue W104 in the crystal structure of the wild-type complex, which is recognized as an important structural requirement for restoring binding affinity. Despite numerous pockets on the protein surface of the mutant hGH–hGHbp complex, the binding-site cavity presents the energetically favorable hot spot for the benzimidazole-containing inhibitors, whereas for a set of nonactive molecules, the lowest energy ligand conformations do not necessarily bind in the engineered cavity. The results reveal a dominant role of the intermolecular van der Waals interactions in providing favorable ligand–protein energetics in the redesigned interface, in agreement with the experimental and computational alanine scanning of the hGH–hGHbp complex.
      <br><br></a_abstract>

    J. Wang, G.M.Verkhivker.
    <a href= "http://journals.aps.org/prl/abstract/10.1103/PhysRevLett.90.188101"> Energy landscape theory, funnels, specificity, and optimal criterion  of biomolecular binding.</a>
    Phys. Rev. Lett. 90: 188101-188104, 2003.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=12786043"> PUBMED. </a>
    <a href="#abstractEner" id="a6">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractEner" id="b6"> 
      ABSTRACT: We study the nature of biomolecular binding. We found that in general there exists several thermodynamic phases: a native binding phase, a non-native phase, and a glass or local trapping phase. The quantitative optimal criterion for the binding specificity is found to be the maximization of the ratio of the binding transition temperature versus the trapping transition temperature, or equivalently the ratio of the energy gap of binding between the native state and the average non-native states versus the dispersion or variance of the non-native states. This leads to a funneled binding energy landscape.
      <br><br></a_abstract>

    G. M. Verkhivker, D. Bouzida, D.K.Gehlhaar, P.A. Rejto, S.T. Freer, P.W. Rose.
    <a href= "http://www.ncbi.nlm.nih.gov/pmc/articles/PMC154313/pdf/pq0903005148.pdf"> Simulating  disorder-order transitions in molecular recognition of unstructured protein where folding meets binding.</a>
    Proc. Natl. Acad. Sci. U.S.A. 100: 5148-5153, 2003.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=12697905"> PUBMED. </a>
    <a href="#abstractSim" id="a7">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractSim" id="b7"> 
      ABSTRACT: A microscopic study of functional disorder–order folding transitions coupled to binding is performed for the p27 protein, which derives a kinetic advantage from the intrinsically disordered unbound form on binding with the phosphorylated cyclin A-cyclindependent kinase 2 (Cdk2) complex. Hierarchy of structural loss during p27 coupled unfolding and unbinding is simulated by using high-temperature Monte Carlo simulations initiated from the crystal structure of the tertiary complex. Subsequent determination of the transition-state ensemble and the proposed atomic picture of the folding mechanism coupled to binding provide a microscopic rationale that reconciles the initiation recruitment of p27 at the cyclin A docking site with the kinetic benefit for a disordered -helix in the unbound form of p27. The emerging structural polarization in the ensemble of unfoldingunbinding trajectories and in the computationally determined transition-state ensemble is not determined by the intrinsic folding preferences of p27 but rather is attributed to the topological requirements of the native intermolecular interface to order -hairpin and -strand of p27 that could be critical for nucleating rapid folding transition coupled to binding. In agreement with the experimental data, the disorder– order folding transition for p27 is largely determined by the functional requirement to form a specific intermolecular interface that ultimately dictates the folding mechanism and overwhelms any local folding preferences for creating a stable -helix in the p27 structure before overcoming the major free energy barrier.
      <br><br></a_abstract>

     
     
    <p class="sub_ctrXS">2002</p>
    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S.T. Freer, P. W. Rose.
    <a href= "http://onlinelibrary.wiley.com/doi/10.1002/prot.10164/abstract"> Monte Carlo simulations of the peptide binding at the consensus binding site of the constant fragment of human immunoglobulin G: the energy landscape analysis of a hot spot at the intermolecular interface.</a>
    Proteins: Struct. Funct. Genet. 48:539-557, 2002.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=12112677"> PUBMED. </a>
    <a href="#abstractMonte" id="a8">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractMonte" id="b8"> 
      ABSTRACT: Monte Carlo simulations of molecular recognition at the consensus binding site of the constant fragment (Fc) of human immunoglobulin G (Ig) protein have been performed to analyze structural and thermodynamic aspects of binding for the 13-residue cyclic peptide DCAWHLGELVWCT. The energy landscape analysis of a hot spot at the intermolecular interface using alanine scanning and equilibrium-simulated tempering dynamics with the simplified, knowledge-based energy function has enabled the role of the protein hot spot residues in providing the thermodynamic stability of the native structure to be determined. We have found that hydrophobic interactions between the peptide and the Met-252, Ile-253, His-433, and His-435 protein residues are critical to guarantee the thermodynamic stability of the crystallographic binding mode of the complex. Binding free energy calculations, using a molecular mechanics force field and a solvation energy model, combined with alanine scanning have been conducted to determine the energetic contribution of the protein hot spot residues in binding affinity. The conserved Asn-434, Ser-254, and Tyr-436 protein residues contribute significantly to the binding affinity of the peptide–protein complex, serving as an energetic hot spot at the intermolecular interface. The results suggest that evolutionary conserved hot spot protein residues at the intermolecular interface may be partitioned in fulfilling thermodynamic stability of the native binding mode and contributing to the binding affinity of the complex. 
      <br><br></a_abstract>

    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S.T. Freer, P. W. Rose.
    <a href= "http://www.sciencedirect.com/science/article/pii/S0959440X0200310X"> Complexity and simplicity of ligand-macromolecule interactions: the energy landscape perspective.</a>
    Curr.  Opin.  Struct.  Biol.  12:197-203, 2002.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=11959497"> PUBMED. </a>
    <a href="#abstractComplex" id="a9">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComplex" id="b9"> 
      ABSTRACT: The energy landscape approach has contributed to recent progress in understanding the complexity and simplicity of ligand–macromolecule interactions. Significant advances in computational structure prediction of ligand–protein complexes have been made using approaches that include the effects of protein flexibility and incorporate a hierarchy of energy functions. The results suggest that the complexity of structure prediction in molecular recognition may be determined by low-resolution properties of the underlying binding energy landscapes and by the nature of the energy funnels near the native structures of the complexes.
      <br><br></a_abstract>

     
     

    <p class="sub_ctrXS">2001</p>
    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S. Arthurs, A. B. Colson,S.T. Freer, V. Larson, B. A. Luty, T. Marrone,  P. W. Rose.
    <a href= "http://www.sciencedirect.com/science/article/pii/S0009261401001610"> Navigating ligand--protein binding energy landscapes: universality and diversity of protein folding and molecular recognition mechanisms.</a>
    Chem. Phys. Lett.  336(5-6): 495-503, 2001.
    <a href="#abstractNav" id="a10">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractNav" id="b10"> 
      ABSTRACT: Thermodynamic and kinetic aspects of ligand–protein binding are studied for the methotrexate–dihydrofolate reductase system from the binding free energy profile constructed as a function of the order parameter. Thermodynamic stability of the native complex and a cooperative transition to the unique native structure suggest the nucleation kinetic mechanism at the equilibrium transition temperature. Structural properties of the transition state ensemble and the ensemble of nucleation conformations are determined by kinetic simulations of the transmission coefficient and ligand–protein association pathways. Structural analysis of the transition states and the nucleation conformations reconciles different views on the nucleation mechanism in protein folding.
      <br><br></a_abstract>

    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S. Arthurs, A. B. Colson,S.T. Freer, V. Larson, B. A. Luty, T. Marrone, P. W. Rose.
    <a href= "http://www.sciencedirect.com/science/article/pii/S0009261401001683"> Parallel simulated tempering dynamics of ligand-protein binding with ensembles of protein conformations.</a>
    Chem. Phys. Lett. 337 (1-3): 181-189, 2001.
    <a href="#abstractPar" id="a11">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractPar" id="b11"> 
      ABSTRACT: Simulated tempering dynamics with the simplified energy model and the ensemble of protein conformations have been performed for the SB203386 inhibitor binding with HIV-1 protease. Equilibrium simulations with multiple protein conformations implicitly incorporate protein flexibility and rank HIV-1 protease conformations according to the average ligand–protein interaction energies. Subsequent energy refinement with a molecular mechanics force field accurately quantifies the energetics of the low-energy ligand binding modes. The results suggest that the mobility of the SB203386 inhibitor is effectively restricted to two symmetry-related binding modes and this may prevent the inhibitor from adapting to distorted binding sites in mutant conformations.
      <br><br></a_abstract>

    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S. Arthurs, A. B. Colson, S.T. Freer, V. Larson, B. A. Luty, T. Marrone,  P. W. Rose.
    <a href="http://pubs.acs.org/doi/abs/10.1021/jm0101141"> Hierarchy of simulation models in predicting structure and energetics of the Src SH2 domain binding to the tyrosyl phosphopeptides.</a>
    J. Med. Chem.  45(1): 72-89, 2001.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=11754580"> PUBMED. </a>
    <a href="#abstractHi" id="a12">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractHi" id="b12"> 
      ABSTRACT: Structure and energetics of the Src Src Homology 2 (SH2) domain binding with the recognition phosphopeptide pYEEI and its mutants are studied by a hierarchical computational approach. The proposed structure prediction strategy includes equilibrium sampling of the peptide conformational space by simulated tempering dynamics with the simplified, knowledge-based energy function, followed by structural clustering of the resulting conformations and binding free energy evaluation of a single representative from each cluster, a cluster center. This protocol is robust in rapid screening of low-energy conformations and recovers the crystal structure of the pYEEI peptide. Thermodynamics of the peptide−SH2 domain binding is analyzed by computing the average energy contributions over conformations from the clusters, structurally similar to the predicted peptide bound structure. Using this approach, the binding thermodynamics for a panel of studied peptides is predicted in a better agreement with the experiment than previously suggested models. However, the overall correlation between computed and experimental binding affinity remains rather modest. The results of this study show that small differences in binding free energies between the Ala and Gly mutants of the pYEEI peptide are considerably more difficult to predict than the structure of the bound peptides, indicating that accurate computational prediction of binding affinities still remains a major methodological and technical challenge.
      <br><br></a_abstract>

    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S. Arthurs, A. B. Colson,S.T. Freer, V. Larson, B. A. Luty, T. Marrone, P. W. Rose.
    <a href= "http://onlinelibrary.wiley.com/doi/10.1002/prot.10019/abstract"> Hierarchy of simulation models in predicting  molecular recognition mechanisms from the binding energy landscapes. Structural analysis of the peptide complexes with SH2 domains.</a>
    Proteins: Struct. Funct. Genet. 45 (4): 456-470, 2001.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=11746693"> PUBMED. </a>
    <a href="#abstractHi" id="a13">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractHi" id="b13"> 
      ABSTRACT: Computer simulations using the simplified energy function and simulated tempering dynamics have accurately determined the native structure of the pYVPML, SVLpYTAVQPNE, and SPGEpYVNIEF peptides in the complexes with SH2 domains. Structural and equilibrium aspects of the peptide binding with SH2 domains have been studied by generating temperature-dependent binding free energy landscapes. Once some native peptide–SH2 domain contacts are constrained, the underlying binding free energy profile has the funnel-like shape that leads to a rapid and consistent acquisition of the native structure. The dominant native topology of the peptide–SH2 domain complexes represents an extended peptide conformation with strong specific interactions in the phosphotyrosine pocket and hydrophobic interactions of the peptide residues C-terminal to the pTyr group. The topological features of the peptide–protein interface are primarily determined by the thermodynamically stable phosphotyrosyl group. A diversity of structurally different binding orientations has been observed for the amino-terminal residues to the phosphotyrosine. The dominant native topology for the peptide residues carboxy-terminal to the phosphotyrosine is tolerant to flexibility in this region of the peptide–SH2 domain interface observed in equilibrium simulations. The energy landscape analysis has revealed a broad, entropically favorable topology of the native binding mode for the bound peptides, which is robust to structural perturbations. This could provide an additional positive mechanism underlying tolerance of the SH2 domains to hydrophobic conservative substitutions in the peptide specificity region.
      <br><br></a_abstract>

    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S. Arthurs, A. B. Colson, S.T. Freer, V. Larson, B. A. Luty, T. Marrone and P. W. Rose.
    <a href= "https://books.google.com/books?id=1VifwQp_crIC&pg=PA157&lpg=PA157&dq=gennady+verkhivker+Binding+energy+landscapes+of+ligand-protein+complexes&source=bl&ots=CEpwGTDKia&sig=-V9MNV46VnllhCprNQcVp8WJfn4&hl=en&sa=X&ved=0ahUKEwjZ6KXMv83RAhXplVQKHamWBwwQ6AEIMzAF#v=onepage&q=gennady%20verkhivker%20Binding%20energy%20landscapes%20of%20ligand-protein%20complexes&f=false">Binding energy landscapes of ligand--protein complexes and molecular docking : Principles, methods and validation experiments. In  &quot;Combinatorial library design and evaluation: principles, software tools, and applications in drug discovery&quot;.</a>
    Edited by Arup K. Ghose and Vellakrad N. Viswanadhan,  p.157-195, Marcel Dekker, New York,  2001.
    <a href="#abstractBin" id="a14">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractBin" id="b14"> 
      ABSTRACT: Deciphering of the common successes and failures in ligand-protein docking, revealed by using different docking strategies and energy functions, allows one to establish fundamental connections between the topology of binding energy landscapes and the results of docking simulations. Binding energy landscape analysis is presented for the methotrexate-dihydrofolate reductase system, which represents a common success in molecular docking of ligand-protein complexes, and for the cyclodextrin glycosyltransferase-maltose complex, which is an example of a common failure in molecular docking simulations. The successes and failures in docking simulations are explained based on the thermodynamic properties determined from equilibrium simulations and the shape of the underlying binding energy landscape. The robust topology of the native structure is a decisive factor contributing to the thermodynamics and dynamics of the methotrexate--dihydrofolate reductase (MTX-DHFR) system that appear to be robust to structural perturbations, variations in the ligand composition, and accuracy of the energetic model. A hierarchical approach that involves a hierarchy of energy functions in proposed in the analysis of a common failure in molecular docking. A protocol of identifying clusters of structurally similar low-energy conformations, generated in equilibrium simulations with the simplified energy function, and subsequent energy minimization with the molecular mechanics force field, resolves a typical common failure in molecular docking.
      <br><br></a_abstract>

    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S. Arthurs, A. B. Colson,S.T. Freer, V. Larson, B. A. Luty, T. Marrone and P. W. Rose.
    <a href= "https://www.researchgate.net/publication/229318345_Chapter_8_Monte_Carlo_simulations_of_HIV-1_protease_binding_dynamics_and_thermodynamics_with_ensembles_of_protein_conformations_Incorporating_protein_flexibility_in_deciphering_mechanisms_of_molecular"> Monte Carlo simulations of HIV-1 protease binding dynamics and thermodynamics with ensembles of protein conformations: incorporating protein flexibility in deciphering mechanisms of molecular recognition. </a>
    In Theoretical and Computational Chemistry  volume 9, “Theoretical Biochemistry- Processes and properties of biological systems&quot;, chapter 8, p.289-340, Elsevier Science, Leiden 2001.
    <a href="#abstractMonte2" id="a15">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractMonte2" id="b15"> 
      ABSTRACT: 
      <br><br></a_abstract>

    Turyanskaya, A. M.; Novikov, A. N.; Verkhivker, G. M.; Kuznetsov, V. V.
    <a href= "http://link.springer.com/article/10.1023%2FA%3A1013930725137"> Conformational composition of 5-alkyl-1,3-oxathianes.</a>
    Russian Journal of General Chemistry (Translation of Zhurnal Obshchei Khimii)  71(9): 1487-1490, 2001.
    <a href="#abstractCon" id="a16">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractCon" id="b16"> 
      ABSTRACT: Conformational equilibrium of 5-isopropyl-1,3-oxathiane occurs mainly between the forms of chair with equatorial and axial orientation of the substituent at the C5 atom, and in the case of 2,2,5-trimethyl- and, apparently, in the case of 5-tert-butyl-1,3-oxathianes is characterized by a more noticeable contribution of flexible forms.
      <br><br></a_abstract>

     
     
    <p class="sub_ctrXS">2000</p>
    G. M. Verkhivker, D. Bouzida, D. K. Gehlhaar, P.A. Rejto, S. Arthurs, A. B. Colson, S.T. Freer, V. Larson, B. A. Luty, T. Marrone,  P. W. Rose.
    <a href= "http://catbert.cs.duke.edu/~brd/Teaching/Bio/asmb/current/Papers/Lit4Docking/MistakeDocking.pdf"> Deciphering common failures in molecular docking of ligand--protein complexes.</a>
    J. Comput. Aided Mol. Des. 14 (8):731-751, 2000.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=11131967"> PUBMED. </a>
    <a href="#abstractHi" id="a17">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractHi" id="b17"> 
      ABSTRACT: Common failures in predicting crystal structures of ligand-protein complexes are investigated for three ligandprotein systems by a combined thermodynamic and kinetic analysis of the binding energy landscapes. Misdocked predictions in ligand-protein docking are classified as ‘soft’ and ‘hard’ failures. While a soft failure arises when the search algorithm is unable to find the global energy minimum corresponding to the crystal structure, a hard failure results from a flaw of the energy function to qualify the crystal structure as the predicted lowest energy conformation in docking simulations. We find that neither the determination of a single structure with the lowest energy nor finding the most common binding mode is sufficient to predict crystal structures of the complexes, which belong to the category of hard failures. In a proposed hierarchical approach, structural similarity clustering of the conformations, generated from equilibrium simulations with the simplified energy function, is followed by energy refinement with the AMBER force field. This protocol, that involves a hierarchy of energy functions, resolves some common failures in ligand-protein docking and detects crystallographic binding modes that were not found during docking simulations.
      <br><br></a_abstract>

     
   
   <p class="sub_ctrXS">More Publications</p> 
    
     <a href="/Publications/2015o2019.php">2015 - 2019 (14 Publications) </a><br>
     <a href="/Publications/2010o2014.php">2010 - 2014 (17 Publications)</a><br>
     <a href="/Publications/2005o2009.php">2005 - 2009 (26 Publications) </a><br>
     <a href="/Publications/1995o1999.php">1995 - 1999 (21 Publications) </a><br>
     <a href="/Publications/1990o1994.php">1990 - 1994 (9 Publications)</a><br>
     <a href="/Publications/o1990.php">Before 1990     (15 Publications) </a><br><br>
   <!-- InstanceEndEditable -->
      <div class="clear"></div>
    
    
    
    </div>
  </div>
  <footer >
    <div id="bottom">
      <div class="clear"></div>
    </div>
  </footer> <div id="subfooter">©2016 Chapman University • One University Drive, Orange, CA 92866 • Phone: (714) 997-6815</div>
</div>
</body>
<!-- InstanceEnd --></html>