<!DOCTYPE html>
<html lang="en"><!-- InstanceBegin template="/Templates/subpage.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<!-- InstanceBeginEditable name="meta" -->
<!-- InstanceEndEditable -->
<!--[if gte IE 9]><meta http-equiv="X-UA-Compatible" content="IE=Edge"><![endif]-->
<!-- InstanceBeginEditable name="doctitle" -->
<title>Chapman University</title>
<!-- InstanceEndEditable -->
<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" type="text/css" href="../css/harv.css">
<link rel="stylesheet" type="text/css" href="../css/harvdrop.css">
<!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><script src="/js/html5shiv.js"></script><![endif]-->
<script src='http://code.jquery.com/jquery-latest.min.js' type='text/javascript'></script><script src="/js/jquery.ui.totop.js"></script>
<!-- InstanceBeginEditable name="head" --><style>
.text_box, p {
	text-align: left !important;
}
#middle #mainsub .biblio-abstract-link.toggle {
}
</style>
<!-- InstanceEndEditable -->
</head>
<body>
<div id="container">
  <div id="smNavBk"><a href="http://compbiosciences.chapman.edu/index.php"><img src="../images/header400light.png" width="332" height="58" alt="Chapman University" style="float: left; margin-left: 10px; margin-top: 10px;"></a>
    <div id="collegeLinks" style="font-size: 24px;">
      <p><a href="https://www.chapman.edu/scst/"><strong><img src="../images/xdz1wyqhwtmxxekidsyy.jpg" width="35" height="35" alt=""/>Schmid College of Science and Technology</strong></a></p>
      <p><a href="http://www.chapman.edu/"><img src="../images/open-graph-cu-logo.png" alt="Chapman University" width="35" height="30" title="Chapman University"/> <span style="font-style: normal; font-size: 18pt;">Chapman University</span></a></p>
    </div>
    <div class="clear"></div>
  </div>
  </header>
  <nav id="topMenu" class="desktop"><!-- #BeginLibraryItem "/Library/topMenu.lbi" --> <nav id="topMenu" class="desktop"><ul id="TopNav" class="MenuBarHorizontal">
      <li> <a href="../index.php">Home</a></li>
      <li><a href="../members.php">Members</a>          
  <li><a href="../Research/research.php" class="MenuBarItemSubmenu">Research</a>
        <ul>
          <li><a href="../Research/computationalCancerBiology.php" >Computational Biology and Translational Bioinformatics in Cancer Research</a></li>
          <li><a href="../Research/proteinKinasesResearch.php" >Protein Kinases</a></li>
          <li><a href="../Research/molecularChaperonesAndCancerResearch.php">Computatoinal Studies of Molecular Chaperones</a></li>
          <li><a href="../Research/translationalBioinformatics.php" >Translational Bioinformatics and Drug Discovery</a></li>
          <li><a href="../Research/relatedResearch.php">Related Links</a></li>
        </ul>
  </li>
      <li><a href="publications.php" class="MenuBarItemSubmenu">Publications</a>          
        <ul>
          <li><a href="2015o2019.php">2015-2019</a></li>
          <li><a href="2010o2014.php">2010-2014</a></li>
          <li><a href="2005o2009.php">2005-2009</a></li>
          <li><a href="2000o2004.php">2000-2004</a></li>
          <li><a href="1995o1999.php">1995-1999</a></li>
          <li><a href="1990o1994.php">1990-1994</a></li>
          <li><a href="o1990.php">Before 1990</a></li>
        </ul>
    <li><a href="../aboutUs.php">About Us</a>  </li>
    <li><a href="../contact.php">Contact Us</a></li>
    </ul>
    <div class="clear"></div></nav><!-- #EndLibraryItem --></nav></div>
  <div id="middle">
    <div id="mainsub"><!-- InstanceBeginEditable name="crumbs" --><script>breadcrumb()</script><!-- InstanceEndEditable -->
     
     
     
    <div class="clear"></div>
    
    
   <!-- InstanceBeginEditable name="main_content_center" -->
    <p class="header">Publications: 2005-2009</p>
   <p class="sub_ctrXS">2009</p>
  Dixit A, Yi L, Gowthaman R, Torkamani A, Schork NJ, Verkhivker GM. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2759519/pdf/pone.0007485.pdf"> Sequence and structure signatures of cancer mutation hotspots in protein kinases. </a> PLoS ONE.  4(10):e7485, 2009.<a href="http://www.ncbi.nlm.nih.gov/pubmed/19834613"> PUBMED.</a><a href="#abstractSequence" id="l1"> Abstract.</a><br><br>

      <a_abstract NAME="abstractSequence" id="t1"><form style="border: 1px solid; height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> Protein kinases are the most common protein domains implicated in cancer, where somatically acquired mutations are known to be functionally linked to a variety of cancers. Resequencing studies of protein kinase coding regions have emphasized the importance of sequence and structure determinants of cancer-causing kinase mutations in understanding of the mutation-dependent activation process. We have developed an integrated bioinformatics resource, which consolidated and mapped all currently available information on genetic modifications in protein kinase genes with sequence, structure and functional data. The integration of diverse data types provided a convenient framework for kinomewide study of sequence-based and structure-based signatures of cancer mutations. The database-driven analysis has revealed a differential enrichment of SNPs categories in functional regions of the kinase domain, demonstrating that a significant number of cancer mutations could fall at structurally equivalent positions (mutational hotspots) within the catalytic core. We have also found that structurally conserved mutational hotspots can be shared by multiple kinase genes and are often enriched by cancer driver mutations with high oncogenic activity. Structural modeling and energetic analysis of the mutational hotspots have suggested a common molecular mechanism of kinase activation by cancer mutations, and have allowed to reconcile the experimental data. According to a proposed mechanism, structural effect of kinase mutations with a high oncogenic potential may manifest in a significant destabilization of the autoinhibited kinase form, which is likely to drive tumorigenesis at some level. Structure-based functional annotation and prediction of cancer mutation effects in protein kinases can facilitate an understanding of the mutation-dependent activation process and inform experimental studies exploring molecular pathology of tumorigenesis.</form><br><br></a_abstract>



    Verkhivker GM, Dixit A, Morra G, Colombo G.<a href="http://www.eurekaselect.com/85322/article"> Structural and computational biology of the molecular chaperone Hsp90: from understanding molecular mechanisms to computer-based inhibitor design.</a>Curr. Top. Med. Chem. 9:1369-1385, 2009. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Structural+and+computational+biology+of+the+molecular+chaperone+Hsp90%3A+from+understanding+molecular+mechanisms+to+computer-based+inhibitor+design.">PUBMED.</a><a href="#abstractStructural" id="l2"> Abstract.</a><br><br>
      <a_abstract NAME="abstractStructural" id ="t2"> ABSTRACT: The molecular chaperone Hsp90 (90 kDa heat-shock protein) mediates many fundamental cellular pathways involved in cell proliferation, cell survival, and cellular stress response. Hsp90 is responsible for the correct conformational development, stability and function in crowded cell environments. Structural and computational biology studies have recently provided important insights into underlying molecular mechanisms of Hsp90 function. These developments have revealed a critical role of Hsp90 structure, conformational dynamics and interdomain communication in promoting the binding and release of ligands and its interaction with client proteins. By disabling multiple signal transduction pathways, Hsp90 inhibition provides a powerful therapeutic strategy in cancer research, which is selective for specific cancer mechanisms, yet broadly applicable to disparate tumors with different genetic signatures. Herein, we review the recent developments in structural and computational studies of Hsp90 function and binding, with the emphasis on progress towards computational structure-based discovery and design of Hsp90 inhibitors. We also review the emerging insights from computational and structure-based approaches to develop anticancer therapies that can target novel allosteric binding sites and Hsp90 interactions with co-chaperones and client proteins. Structural and computational biology studies can provide a foundation for the design of Hsp90 modulators capable of regulating functional protein motions linked to biological activities. We highlight current challenges in translating molecular mechanisms of the molecular chaperone into therapeutic strategies and outline future directions for the computer-based design of Hsp90 inhibitors.<br><br></a_abstract>


    Dixit A, Verkhivker GM.<a href="http://www.ploscompbiol.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1000487&representation=PDF">Hierarchical modeling of activation mechanisms in the ABL and EGFR kinase domains: thermodynamic and mechanistic catalysts of kinase activation by cancer mutations.</a>PLoS Comput. Biol. 5(8):e1000487, 2009.<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Hierarchical+modeling+of+activation+mechanisms+in+the+ABL+and+EGFR+kinase+domains%3A+thermodynamic+and+mechanistic+catalysts+of+kinase+activation+by+cancer+mutations."> PUBMED. </a><a href="#abstractHi" id="l3">Abstract.</a><br><br>
      <a_abstract NAME="abstractHi" id="t3"> ABSTRACT: Structural and functional studies of the ABL and EGFR kinase domains have recently suggested a common mechanism of activation by cancer-causing mutations. However, dynamics and mechanistic aspects of kinase activation by cancer mutations that stimulate conformational transitions and thermodynamic stabilization of the constitutively active kinase form remain elusive. We present a large-scale computational investigation of activation mechanisms in the ABL and EGFR kinase domains by a panel of clinically important cancer mutants ABL-T315I, ABL-L387M, EGFR-T790M, and EGFR-L858R. We have also simulated the activating effect of the gatekeeper mutation on conformational dynamics and allosteric interactions in functional states of the ABL-SH2-SH3 regulatory complexes. A comprehensive analysis was conducted using a hierarchy of computational approaches that included homology modeling, molecular dynamics simulations, protein stability analysis, targeted molecular dynamics, and molecular docking. Collectively, the results of this study have revealed thermodynamic and mechanistic catalysts of kinase activation by major cancer-causing mutations in the ABL and EGFR kinase domains. By using multiple crystallographic states of ABL and EGFR, computer simulations have allowed one to map dynamics of conformational fluctuations and transitions in the normal (wild-type) and oncogenic kinase forms. A proposed multi-stage mechanistic model of activation involves a series of cooperative transitions between different conformational states, including assembly of the hydrophobic spine, the formation of the Src-like intermediate structure, and a cooperative breakage and formation of characteristic salt bridges, which signify transition to the active kinase form. We suggest that molecular mechanisms of activation by cancer mutations could mimic the activation process of the normal kinase, yet exploiting conserved structural catalysts to accelerate a conformational transition and the enhanced stabilization of the active kinase form. The results of this study reconcile current experimental data with insights from theoretical approaches, pointing to general mechanistic aspects of activating transitions in protein kinases.<br><br></a_abstract>

     Zheng K, Laurence JS, Kuczera K, Verkhivker G, Middaugh CR, Siahaan TJ.
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3328966/pdf/nihms162557.pdf"> Characterization of multiple stable conformers of the EC5 domain of E-cadherin and the interaction of EC5 with E-cadherin peptides. </a>
    Chem. Bio.l Drug Des. 73(6):584-598, 2009.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Characterization+of+multiple+stable+conformers+of+the+EC5+domain+of+E-cadherin+and+the+interaction+of+EC5+with+E-cadherin+peptides."> PUBMED. </a>
    <a href="#abstractChar" id="l41">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractChar" id="t41"> 
      ABSTRACT: The objectives of this work were to express the EC5 domain of E-cadherin and determine its structural characteristics as well as to evaluate the binding properties of HAV and BLG4 peptides to EC5 using spectroscopic methods. Homophilic interactions of E-cadherins are responsible for cell-cell adhesion in the adherens junctions of the biological barriers (i.e., intestinal mucosa and blood-brain barriers). The EC5 domain of E-cadherin has an important role in T-cell adhesion to intestinal mucosa via αEβ7 integrin-E-cadherin interactions. In this study, the expressed EC5 has a high thermal stability (Tm = 64.3 °C); it also has two stable conformations at room temperature, which convert to one conformation at approximately 54.5 °C. NMR and FTIR showed that HAV and BLG4 peptides bind to EC5. HSQC-NMR showed that either Asn or Gln of EC5 was involved in the interactions with HAV and BLG4 peptides. EC5 underwent a conformational change upon interaction with the HAV and BLG4 peptides. Finally, the binding properties of both peptides were modeled by docking experiments, and the results suggest that Asn-46 and Asn-75 of EC5 could be involved during the interaction with the peptides and that the Ser and Trp residues of the HAV and BLG4 peptides, respectively, were important for binding to EC5.  
      <br><br></a_abstract>

    Morra G, Verkhivker G, Colombo G.
    <a href= "http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2649446/pdf/pcbi.1000323.pdf">Modeling signal propagation mechanisms and ligand-based conformational dynamics of the Hsp90 molecular chaperone full-length dimer.</a>
    PLoS Comput. Biol. 5(3):e1000323, 2009.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Modeling+signal+propagation+mechanisms+and+ligand-based+conformational+dynamics+of+the+Hsp90+molecular+chaperone+full-length+dimer."> PUBMED. </a>
    <a href="#abstractMod" id="l4">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractMod" id="t4"> 
      ABSTRACT: Hsp90 is a molecular chaperone essential for protein folding and activation in normal homeostasis and stress response. ATP binding and hydrolysis facilitate Hsp90 conformational changes required for client activation. Hsp90 plays an important role in disease states, particularly in cancer, where chaperoning of the mutated and overexpressed oncoproteins is important for function. Recent studies have illuminated mechanisms related to the chaperone function. However, an atomic resolution view of Hsp90 conformational dynamics, determined by the presence of different binding partners, is critical to define communication pathways between remote residues in different domains intimately affecting the chaperone cycle. Here, we present a computational analysis of signal propagation and long-range communication pathways in Hsp90. We carried out molecular dynamics simulations of the full-length Hsp90 dimer, combined with essential dynamics, correlation analysis, and a signal propagation model. All-atom MD simulations with timescales of 70 ns have been performed for complexes with the natural substrates ATP and ADP and for the unliganded dimer. We elucidate the mechanisms of signal propagation and determine ‘‘hot spots’’ involved in interdomain communication pathways from the nucleotide-binding site to the C-terminal domain interface. A comprehensive computational analysis of the Hsp90 communication pathways and dynamics at atomic resolution has revealed the role of the nucleotide in effecting conformational changes, elucidating the mechanisms of signal propagation. Functionally important residues and secondary structure elements emerge as effective mediators of communication between the nucleotide-binding site and the C-terminal interface. Furthermore, we show that specific interdomain signal propagation pathways may be activated as a function of the ligand. Our results support a ‘‘conformational selection model’’ of the Hsp90 mechanism, whereby the protein may exist in a dynamic equilibrium between different conformational states available on the energy landscape and binding of a specific partner can bias the equilibrium toward functionally relevant complexes.
      <br><br></a_abstract>

    Trivedi M, Davis RA, Shabaik Y, Roy A, Verkhivker G, Laurence JS, Middaugh CR, Siahaan TJ.
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3319692/pdf/nihms362424.pdf">The role of covalent dimerization on the physical and chemical  stability of the EC1 domain of human E-cadherin.</a>
    J. Pharm. Sci. 98(10):3562-3574, 2009.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=The+role+of+covalent+dimerization+on+the+physical+and+chemical+stability+of+the+EC1+domain+of+human+E-cadherin."> PUBMED. </a>
    <a href="#abstractRol" id="l5">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractRol" id="t5"> 
      ABSTRACT: The objective of this work was to evaluate the solution stability of the EC1 domain of E-cadherin under various conditions. The EC1 domain was incubated at various temperatures (4, 37, and 70 °C) and pH values (3.0, 7.0, and 9.0). At pH 9.0 and 37 or 70 °C, a significant loss of EC1 was observed due to precipitation and a hydrolysis reaction. The degradation was suppressed upon addition of DTT, suggesting that the formation of EC1 dimer facilitated the EC1 degradation. At 4 °C and various pH values, the EC1 secondary and tertiary showed changes upon incubation up to 28 days, and DTT prevented any structural changes upon 28 days of incubation. Molecular dynamics simulations indicated that the dimer of EC1 has higher mobility than does the monomer; this higher mobility of the EC1 dimer may contribute to instability of the EC1 domain.
      <br><br></a_abstract>

    Dixit A, Torkamani A, Schork NJ, Verkhivker G.
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2716637/pdf/main.pdf">Computational modeling of structurally conserved cancer mutations in the RET and MET kinases: the impact on protein structure, dynamics, and stability.</a>
    Biophys J. 96(3):858-874, 2009.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19186126"> PUBMED. </a>
    <a href="#abstractComp" id="l6">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComp" id="t6"> 
      ABSTRACT: Structural and biochemical characterization of protein kinases that confer oncogene addiction and harbor a large number of disease-associated mutations, including RET and MET kinases, have provided insights into molecular mechanisms associated with the protein kinase activation in human cancer. In this article, structural modeling, molecular dynamics, and free energy simulations of a structurally conserved mutational hotspot, shared by M918T in RET and M1250T in MET kinases, are undertaken to quantify the molecular mechanism of activation and the functional role of cancer mutations in altering protein kinase structure, dynamics, and stability. The mechanistic basis of the activating RET and MET cancer mutations may be driven by an appreciable free energy destabilization of the inactive kinase state in the mutational forms. According to our results, the locally enhanced mobility of the cancer mutants and a higher conformational entropy are counterbalanced by a larger enthalpy loss and result in the decreased thermodynamic stability. The computed protein stability differences between the wild-type and cancer kinase mutants are consistent with circular dichroism spectroscopy and differential scanning calorimetry experiments. These results support the molecular mechanism of activation, which causes a detrimental imbalance in the dynamic equilibrium shifted toward the active form of the enzyme. Furthermore, computer simulations of the inhibitor binding with the oncogenic and drug-resistant RET mutations have also provided a plausible molecular rationale for the observed differences in the inhibition profiles, which is consistent with the experimental data. Finally, structural mapping of RET and MET cancer mutations and the computed protein stability changes suggest a similar mechanism of activation, whereby the cancer mutations which display the higher oncogenic activity tend to have the greatest destabilization effect on the inactive kinase structure.
      <br><br></a_abstract>

    Torkamani A, Verkhivker G, Schork NJ.
    <a href= "http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2905872/pdf/nihms130046.pdf">Cancer driver mutations in protein kinase genes.</a>
    Cancer Lett.  281(2):117-127, 2009.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/19081671"> PUBMED. </a>
    <a href="#abstractCan" id="l7">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractCan" id="t7"> 
      ABSTRACT: Recent studies investigating the genetic determinants of cancer suggest that some of the genetic alterations contributing to tumorigenesis may be inherited, but the vast majority are somatically acquired during the transition of a normal cell to a cancer cell. A systematic understanding of the genetic and molecular determinants of cancers has already begun to have a transformative effect on the study and treatment of cancer, particularly through the identification of a range of genetic alterations in protein kinase genes, which are highly associated with the disease. Since kinases are prominent therapeutic targets for intervention within the cancer cell, studying the impact that genomic alterations within them have on cancer initiation, progression, and treatment is both logical and timely. In fact, recent sequencing and resequencing (i.e., polymorphism idenitification) efforts have catalyzed the quest for protein kinase ‘driver’ mutations (i.e., those genetic alterations which contribute to the transformation of a normal cell to a proliferating cancerous cell) in distinction to kinase ‘passenger’ mutations which reflect mutations that merely build up in course of normal and unchecked (i.e., cancerous) somatic cell replication and proliferation. In this review, we discuss the recent progress in the discovery and functional characterization of protein kinase cancer driver mutations and the implications of this progress for understanding tumorigenesis as well as the design of ‘personalized’ cancer therapeutics that target an individual’s unique mutational profile. 
      <br><br></a_abstract>

    Verkhivker G.
    <a href="http://www.aiimjournal.com/article/S0933-3657(08)00125-5/pdf">Computational proteomics analysis of binding mechanisms and molecular signatures of the HIV-1 protease drugs.</a>
    Artif. Intell. Med. 45:197-206, 2009.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18926674"> PUBMED. </a>
    <a href="#abstractCompu" id="l8">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractCompu" id="t8"> 
      ABSTRACT: Computational proteomics analysis of biomolecular interactions is proposed to determine molecular signatures of the HIV-1 protease inhibitors. A comparative microscopic analysis is conducted for a panel of inhibitors which exemplify a diversity of the HIV-1 PR binding mechanisms, from the active site inhibition to intervening with the protease folding and dimerization.
      <br><br></a_abstract>


<p class="sub_ctrXS">2008</p>
    G.M. Verkhivker.
    <a href="http://link.springer.com/chapter/10.1007%2F978-3-642-02504-4_1">Coarse-grained modeling  of the HIV--1 protease binding  mechanisms: I. Targeting structural flexibility of the protease flaps and implications for drug design.</a>
    Lect. Notes Comput. Sci.  LNBI 5488, p.1-12, 2008.
    <a href="#abstractCoar" id="l9">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractCoar" id="t9"> 
      ABSTRACT: We propose a coarse–grained model to study binding mechanism of the HIV–1 protease inhibitors using long equilibrium simulations with an ensemble of the HIV–1 protease crystal structures. A microscopic analysis suggests a binding mechanism, in which the HIV–1 protease drugs may exploit the dynamic equilibrium between thermodynamically stable, high affinity complexes with the closed form of the HIV–1 protease and meta–stable intermediate complexes with the alternative structural forms of the protease. We have found that formation of the hydrophobic interaction clusters with the conserved flap residues may stabilize semi–open and open forms of the enzyme and lead to weakly bound, transient inhibitor complexes. The results suggest that inhibitors may function through a multi-mechanistic effect of stabilizing structurally different conformational states of the protease, highlighting the molecular basis of the flap residues in developing drug resistance.
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href="http://link.springer.com/chapter/10.1007/978-3-642-02504-4_2">Coarse-grained modeling  of the HIV--1 protease binding  mechanisms: II. Folding inhibition.</a>
    Lect. Notes Comput. Sci.  LNBI 5488, p.13-24, 2008.
    <a href="#abstractCoar2" id="l10">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractCoar2" id="t10"> 
      ABSTRACT: Evolutionary and structurally conserved fragments 24–34 and 83–93 from each of the HIV–1 protease (HIV–1 PR) monomers constitute the critical components of the HIV–1 PR folding nucleus. It has been recently discovered that the peptide with the amino acid sequence NIIGRNLLTQI identical to the corresponding segment 83–93 of the HIV–1 PR monomer, can inhibit folding of HIV–1 PR. We have previously shown that this peptide can form stable complexes with the folded HIV–1 PR monomer by targeting the conserved segment 24–34 of the folding nucleus (folding inhibition) and by interacting with the antiparallel termini β–sheet region (dimerization inhibition). In this follow-up study, we propose a generalized, coarse–grained model of the folding inhibition based simulations with an ensemble of both folded and partially unfolded HIV–1 PR conformational states. Using a dynamic equilibrium between low–energy complexes formed with the folded and partially unfolded HIV–1 PR monomers, the NIIGRNLLTQI peptide may effectively intervene with the HIV–1 PR folding and dimerization. The performed microscopic analysis reconciles the experimental and computational results and rationalizes the molecular basis of folding inhibition.
      <br><br></a_abstract>   

    Iskandarsyah, Tejo BA, Tambunan US, Verkhivker G, Siahaan TJ. 
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519170/pdf/nihms60436.pdf">Structural modifications of ICAM-1 cyclic peptides to improve the activity to inhibit heterotypic adhesion of T cells.</a>
    Chem. Biol. Drug Des. 72(1):27-33, 2008.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18554252"> PUBMED. </a>
    <a href="#abstractStruct2" id="l11">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractStruct2" id="t11"> 
      ABSTRACT: LFA-1/ICAM-1 interaction plays an important role in the formation of the immunological synapse between T cells and antigen-presenting cells (APC). Blocking of LFA-1/ICAM-1 interactions has been shown to suppress the progression of autoimmune diseases. cIBR peptide (cyclo(1,12)PenPRGGSVLVTGC) inhibits ICAM-1/LFA-1 interaction by binding to the I-domain of LFA-1. To increase the bioactivity of cIBR peptide, we systemically modified the structure of the peptide by (a) replacing the Pen residue at the N-terminus with Cys, (b) cyclization using amide bond formation between Lys-Glu side chains, and (c) reducing the peptide size by eliminating the C-terminal residue. We found that the activity of cIBR peptide was not affected by replacing Phe with Cys. Peptide cyclization by forming the Lys-Glu amide bond also increased the activity of cIBR peptide, presumably due to the resistance of the amide bond to the reducing nature of glutathione in plasma. We also found that a reduced derivative of cIBR with eight residues (cyclo(1,8)CPRGGSVC) has a bioactivity similar to that of the larger cIBR peptides. Our findings suggest that, by systemically modifying the structure of cIBR peptide, the biological activity of these derivatives can be optimized for future use to inhibit T-cell adhesion in in vivo models of autoimmune diseases.
      <br><br></a_abstract>

    Colombo G, Morra G, Meli M, Verkhivker G. 
    <a href= "http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2402385/pdf/zpq7976.pdf">Understanding ligand-based modulation of the Hsp90 molecular  chaperone dynamics at atomic resolution.</a>
    Proc. Natl. Acad. Sci. U. S. A. 105:7976-7981, 2008.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18511558"> PUBMED. </a>
    <a href="#abstractUnder" id="l12">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractUnder" id="t12"> 
      ABSTRACT: Molecular switching and ligand-based modulation of the 90-kDa heat-shock protein (Hsp90) chaperone activity may ultimately facilitate conformational coupling to the ATPase cycle along with activation and recruitment of the broad range of client proteins. We present an atomic resolution analysis of the Hsp90 N-terminal domain (NTD) binding energy landscape by simulating protein dynamics with a range of binding partners. We show that the activity of the molecular chaperone may be linked to (i) local folding-unfolding transitions and conformational switching of the ‘‘active site lid’’ upon binding and (ii) differences in the underlying protein dynamics as a function of the binding partner. This study suggests that structural plasticity of the Hsp90 NTD can be exploited by the molecular chaperone machinery to modulate enhanced structural rigidity during ATP binding and increased protein flexibility as a consequence of the inhibitor binding. The present study agrees with the experimental structural data and provides a plausible molecular model for understanding mechanisms of modulation of molecular chaperone activities by binding partners.
      <br><br></a_abstract>

    G. M. Verkhivker,  Tiana G, Camilloni C, Provasi D, Broglia RA.
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2440460/pdf/550.pdf">Atomistic simulations of the HIV-1 protease folding  inhibition.</a>
    Biophys J. 95:550-562, 2008.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18375506"> PUBMED. </a>
    <a href="#abstractAtom" id="l13">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractAtom" id="t13"> 
      ABSTRACT: Biochemical experiments have recently revealed that the p-S8 peptide, with an amino-acid sequence identical to the conserved fragment 83–93 (S8) of the HIV-1 protease, can inhibit catalytic activity of the enzyme by interfering with protease folding and dimerization. In this study, we introduce a hierarchical modeling approach for understanding the molecular basis of the HIV-1 protease folding inhibition. Coarse-grained molecular docking simulations of the flexible p-S8 peptide with the ensembles of HIV-1 protease monomers have revealed structurally different complexes of the p-S8 peptide, which can be formed by targeting the conserved segment 24–34 (S2) of the folding nucleus (folding inhibition) and by interacting with the antiparallel termini b-sheet region (dimerization inhibition). All-atom molecular dynamics simulations of the inhibitor complexes with the HIV-1 PR monomer have been independently carried out for the predicted folding and dimerization binding modes of the p-S8 peptide, confirming the thermodynamic stability of these complexes. Binding free-energy calculations of the p-S8 peptide and its active analogs are then performed using molecular dynamics trajectories of the peptide complexes with the HIV-1 PR monomers. The results of this study have provided a plausible molecular model for the inhibitor intervention with the HIV-1 PR folding and dimerization and have accurately reproduced the experimental inhibition profiles of the active folding inhibitors.
      <br><br></a_abstract>

     

<p class="sub_ctrXS">2007</p>
    J. Wang, X. Zheng, Y. Yang, D. Drueckhammer, W. Yang,  G. Verkhivker, E. Wang.
    <a href="http://journals.aps.org/prl/abstract/10.1103/PhysRevLett.99.198101">Quantifying intrinsic specificity: A potential complement to affinity in drug screening.</a>
    Phys. Rev. Lett.  99:198101-198104, 2007.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18233118"> PUBMED. </a>
    <a href="#abstractQuant" id="l14">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractQuant" id="t14"> 
      ABSTRACT: We report here the investigation of a novel description of specificity in protein-ligand binding based on energy landscape theory. We define a new term, intrinsic specificity ratio (ISR), which describes the level of discrimination in binding free energies of the native basin for a protein-ligand complex from the weaker binding states of the same ligand. We discuss the relationship between the intrinsic specificity we defined here and the conventional definition of specificity. In a docking study of molecules with the enzyme COX-2, we demonstrate a statistical correspondence between ISR value and geometrical shapes of the small molecules binding to COX-2. We further observe that the known selective (nonselective) inhibitors of COX-2 have higher (lower) ISR values. We suggest that intrinsic specificity ratio may be a useful new criterion and a complement to affinity in drug screening and in searching for potential drug lead compounds.
      <br><br></a_abstract>

    H. Yusuf-Makagiansar, T. V. Yakovleva, B. A. Tejo, K. O. Hamilton, Y. Hu, G.M. Verkhivker, K. L. Audus, T.J. Siahaan.
    <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1747-0285.2007.00549.x/abstract">Sequence recognition of α-LFA-1-derived peptides by ICAM-1 cell-receptors: inhibitors of T-cell adhesion.</a>
    Chem. Biol.  Drug Des. 70:237-246, 2007.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17718718"> PUBMED. </a>
    <a href="#abstractSeq2" id="l15">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractSeq2" id="t15"> 
      ABSTRACT: Blocking the T-cell adhesion signal from intercellular adhesion molecule-1/leukocyte function-associated antigen-1 interactions (Signal-2) can suppress the progression of autoimmune diseases (i.e. type-1 diabetes, psoriasis) and prevent allograph rejection. In this study, we determined the active region(s) of cLAB.L peptide [cyclo(1,12)Pen-ITDGEATDSGC] by synthesizing and evaluating the biologic activity of hexapeptides in inhibiting T-cell adhesion. A new heterotypic T-cell adhesion assay was also developed to provide a model for the T-cell adhesion process during lung inflammation. Two hexapeptides, ITDGEA and DGEATD, were found to be more active than the other linear hexapeptides. The cyclic derivative of ITDGEA [i.e. cyclo(1,6)ITDGEA] has similar activity than the parent linear peptide and has lower activity than cLAB.L peptide. Computational-binding experiments were carried out to explain the possible mechanism of binding of these peptides to intercellular adhesion molecule-1. Both ITDGEA and DGEATD bind the same site on intercellular adhesion molecule-1 and they interact with the Gln34 and Gln73 residues on D1 of intercellular adhesion molecule-1. In the future, more potent derivatives of cyclo(1,6)ITDGEA will be designed by utilizing structural and binding studies of the peptide to intercellular adhesion molecule-1. The heterotypic T-cell adhesion to Calu-3 will also be used as another assay to evaluate the selectivity of the designed peptides.
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href= "http://bioinformatics.oxfordjournals.org/content/23/15/1919.full.pdf">Exploring sequence-structure relationships in the tyrosinekinome space : functional classification of the binding specificity mechanisms for cancer therapeutics.</a>
    Bioinformatics  23:1919-1926, 2007.
    <a href= "http://www.ncbi.nlm.nih.gov/pubmed/?term=17537753"> PUBMED. </a>
    <a href="#abstractExp" id="l16">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractExp" id="t16"> 
      ABSTRACT: Evolutionary and structural conservation patterns shared by more than 500 of identified protein kinases have led to complex sequence-structure relationships of cross-reactivity for kinase inhibitors. Understanding the molecular basis of binding specificity for protein kinases family, which is the central problem in discovery of cancer therapeutics, remains challenging as the inhibitor selectivity is not readily interpreted from chemical proteomics studies, neither it is easily discernable directly from sequence or structure information. We present an integrated view of sequencestructure-binding relationships in the tyrosine kinome space in which evolutionary analysis of the kinases binding sites is combined with computational proteomics profiling of the inhibitor–protein interactions. This approach provides a functional classification of the binding specificity mechanisms for cancer agents targeting protein tyrosine kinases.
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href= "http://onlinelibrary.wiley.com/doi/10.1002/bip.20656/abstract"> In silico profiling of tyrosine kinases binding specificity and drug resistance using Monte Carlo simulations with the ensembles of protein kinase crystal structures.</a>
    Biopolymers 85:333-348, 2007.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17167796"> PUBMED. </a>
    <a href="#abstractSil" id="l17">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractSil" id="t17"> 
      ABSTRACT: The molecular basis of the tyrosine kinases binding specificity and drug resistance against cancer drugs Imatinib and Dasatinib is elucidated using Monte Carlo simulations of the inhibitor–receptor binding with the ensembles of protein kinase crystal structures. In silico proteomics analysis unravels mechanisms by which structural plasticity of the tyrosine kinases is linked with the conformational preferences of Imatinib and Dasatinib in achieving effective drug binding with a distinct spectrum of the tyrosine kinome. The differences in the inhibitor sensitivities to the ABL kinase mutants are rationalized based on variations in the binding free energy profiles with the conformational states of the ABL kinase. While Imatinib binding is highly sensitive to the activation state of the enzyme, the computed binding profile of Dasatinib is remarkably tolerant to the conformational state of ABL. A comparative analysis of the inhibitor binding profiles with the clinically important ABL kinase mutants has revealed an excellent agreement with the biochemical and proteomics data. We have found that conformational adaptability of the kinase inhibitors to structurally different conformational states of the tyrosine kinases may have pharmacological relevance in acquiring a specific array of potent activities and regulating a scope of the inhibitor resistance mutations. This study outlines a useful approach for understanding and predicting the molecular basis of the inhibitor sensitivity against potential kinase targets and drug resistance.
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href= "http://onlinelibrary.wiley.com/doi/10.1002/prot.21287/abstract">Computational proteomics of biomolecular interactions in the sequence and structure space of the tyrosine kinome: deciphering the molecular basis of the kinase inhibitors selectivity.</a>
    Proteins 66:912-929, 2007.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17173284"> PUBMED. </a>
    <a href="#abstractComp3" id="l18">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComp3" id="t18"> 
      ABSTRACT: Understanding and predicting the molecular basis of protein kinases specificity against existing therapeutic agents remains highly challenging and deciphering this complexity presents an important problem in discovery and development of effective cancer drugs. We explore a recently introduced computational approach for in silico profiling of the tyrosine kinases binding specificity with a class of the pyrido-[2,3-d]pyrimidine kinase inhibitors. Computational proteomics analysis of the ligand–protein interactions using parallel simulated tempering with an ensemble of the tyrosine kinases crystal structures reveals an important molecular determinant of the kinase specificity. The pyrido-[2,3-d]pyrimidine inhibitors are capable of dynamically interacting with both active and inactive forms of the tyrosine kinases, accommodating structurally different kinase conformations with a similar binding affinity. Conformational tolerance of the protein tyrosine kinases binding with the pyrido[2,3-d]pyrimidine inhibitors provides the molecular basis for the broad spectrum of potent activities and agrees with the experimental inhibition profiles. The analysis of the pyrido[2,3-d]pyrimidine sensitivities against a number of clinically relevant ABL kinase mutants suggests an important role of conformational adaptability of multitargeted kinase inhibitors in developing drug resistance mechanisms. The presented computational approach may be useful in complementing proteomics technologies to characterize activity signatures of small molecules against a large number of potential kinase targets.
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href="http://ebooks.iospress.nl/volumearticle/26600">Exploring mechanisms of protein folding and binding in signal transduction networks.</a>
    Proceedings of  The International School of Physics “Enrico Fermi” Course CLXV “Protein Folding and Drug Design.”  165: 115-133. Edited by R.A. Broglia, L. Serrano, and G. Tiana, IOS Press, 2007.
    <a href="#abstractExpl" id="l19">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractExpl" id="t19"> 
      ABSTRACT: Proteins from complex interaction networks that determine much of the physiology and function of the cell. The Architecture of protein-protein interaction networks was recently proposed to be scale-free with most of the proteins having only few connections but with relatively fewer protein hubs establishing a considerably large number of links. The presence of scale-free networks and protein hubs in biological systems has indicated that evolutionary processes may exhibit mechanisms of preferential attachment based on duplication and divergence of genes. The organization of protein interaction networks may be imprinted in structural properties of the binding sites which enable network hubs to interact with diverse range of protein systems by incorporating conformational flexibility and even intrinsic disorder in one or both binding partners. Functional promiscuity is often linked with protein conformational diversity, which can be considered as evolvability traits that enable existing enzymes to rapidly develop new activities. When combined with the classical mechanisms of gene duplication mutation and selection and models of divergent evolution, conformational diversity provides a powerful mechanism to facilitate the evolution of new functions. There is growing evidence from direct evolution experiments the proteins with promiscuous functions may have divergently evolved to acquire higher specificity and activity based on their ability to alter functions using sequence plasticity of a relatively small number of amino acid substutions. Numerous biochemical and genomic analysis have suggested that proteins may have developed the ability to improve novel or alter existing functions using plasticity of amino acids residing insode or near active sites. Direct evolution studies have also indicated that substantial changes in the prominscuous functions of a protein need not come at the expense of its native functions as evolution of a new function may be driven by mutations that have a minor effect on the native function but larger effects on the promiscuous functions.
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href="http://ebooks.iospress.nl/volumearticle/26607">Computational structural proteomics of the kinases binding specificity and drug resistance.</a>
    Proceedings of  The International School of Physics “Enrico Fermi” Course CLXV “Protein Folding and Drug Design.”  165: 221-238. Edited by R.A. Broglia, L. Serrano, and G. Tiana, IOS Press, 2007.
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/17173284"> PUBMED. </a>
    <a href="#abstractComp4" id="l20">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComp4" id="t20"> 
      ABSTRACT: Understanding and predicting the molecular basis of protein kinases specificity against existing therapeutic agents remains highly challenging and deciphering this complexity presents an important problem in discovery and development of effective cancer drugs. We explore a recently introduced computational approach for in silico profiling of the tyrosine kinases binding specificity with a class of the pyrido-[2,3-d]pyrimidine kinase inhibitors. Computational proteomics analysis of the ligand-protein interactions using parallel simulated tempering with an ensemble of the tyrosine kinases crystal structures reveals an important molecular determinant of the kinase specificity. The pyrido-[2,3-d]pyrimidine inhibitors are capable of dynamically interacting with both active and inactive forms of the tyrosine kinases, accommodating structurally different kinase conformations with a similar binding affinity. Conformational tolerance of the protein tyrosine kinases binding with the pyrido[2,3-d]pyrimidine inhibitors provides the molecular basis for the broad spectrum of potent activities and agrees with the experimental inhibition profiles. The analysis of the pyrido[2,3-d]pyrimidine sensitivities against a number of clinically relevant ABL kinase mutants suggests an important role of conformational adaptability of multitargeted kinase inhibitors in developing drug resistance mechanisms. The presented computational approach may be useful in complementing proteomics technologies to characterize activity signatures of small molecules against a large number of potential kinase targets.
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href="http://ebooks.iospress.nl/volumearticle/26609">Energy landscapes of  bimolecular binding and molecular modulators of protein--protein interactions.</a>
    Proceedings of  The International School of Physics “Enrico Fermi” Course CLXV “Protein Folding and Drug Design.” 165:253-271. Edited by R.A. Broglia, L. Serrano, and G. Tiana, IOS Press, 2007.
    <a href="#abstractEnerg" id="l21">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractEnerg" id="t21"> 
      ABSTRACT: 
      <br><br></a_abstract>

    G.M. Verkhivker.
    <a href="http://link.springer.com/chapter/10.1007%2F978-3-540-73400-0_77">Computational proteomics of biomolecular interactions in sequence and structure space of the tyrosine kinome: evolutionary constraints and protein conformational selection determine binding signatures of cancer drugs.</a>
    Lecture Notes in  Artificial Intelligence, Subseries Lect. Notes Comput. Sci.  Science, Springer, p.  604-612, 2007.
    <a href="#abstractComp5" id="l22">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractComp5" id="t22"> 
      ABSTRACT: The emerging insights into kinase function and evolution combined with a rapidly growing number of crystal structures of protein kinases complexes have facilitated a comprehensive structural bioinformatics analysis of sequence–structure relationships in determining the binding function of protein tyrosine kinases. We have found that evolutionary signal derived solely from the tyrosine kinase sequence conservation can not be readily translated into the ligand binding phenotype. However, fingerprinting ligand–protein interactions using in silico profiling of inhibitor binding against protein tyrosine kinases crystal structures can detect a functionally relevant kinase binding signal and reconcile the existing experimental data. In silico proteomics analysis unravels mechanisms by which structural plasticity of the tyrosine kinases is linked with the conformational preferences of cancer drugs Imatinib and Dasatinib in achieving effective drug binding with a distinct spectrum of the tyrosine kinome. While Imatinib binding is highly sensitive to the activation state of the enzyme, the computed binding profile of Dasatinib is remarkably tolerant to the conformational state of ABL. A comprehensive study of evolutionary, structural, dynamic and energetic aspects of tyrosine kinases binding with clinically important class of inhibitors provides important insights into mechanisms of sequence–structure relationships in the kinome space and molecular basis of functional adaptability towards specific binding.
      <br><br></a_abstract>


     
  <p class="sub_ctrXS">2006</p>
    G.M. Verkhivker.
    <a href= "http://bioinformatics.oxfordjournals.org/content/22/15/1846.full.pdf">Imprint of evolutionary conservation and protein structure variation on the binding function of protein tyrosine kinases.</a>
    Bioinformatics 22:1846-1854, 2006.
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16720585"> PUBMED. </a>
    <a href="#abstractIm" id="l23">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractIm" id="t23"> 
      ABSTRACT: According to the models of divergent molecular evolution, the evolvability of new protein function may depend on the induction of new phenotypic traits by a small number of mutations of the binding site residues. Evolutionary relationships between protein kinases are often employed to infer inhibitor binding profiles from sequence analysis. However, protein kinases binding profiles may display inhibitor selectivity within a given kinase subfamily, while exhibiting cross-activity between kinases that are phylogenetically remote from the prime target. The emerging insights into kinase function and evolution combined with a rapidly growing number of publically available crystal structures of protein kinases complexes have motivated structural bioinformatics analysis of sequence-structure relationships in determining the binding function of protein tyrosine kinases.
      <br><br></a_abstract>

     

  <p class="sub_ctrXS">2005</p>
    G.M. Verkhivker.
    <a href= LINK1 >Simulating ligand binding in sequence and structure space of the human protein kinome : imprint of protein conformational selection and evolutionary constraints determines binding function.</a>
    In “Algorithms and Computational Methods for Biochemical and Evolutionary Networks: Compbionets-2005”. Edited by M.F. Saqot and K.S. Guimaraes, 164p. College Publications, 2005.
    <a href= LINK2 > PUBMED. </a>
    <a href="#abstractSim" id="l24">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractSim" id="t24"> 
      ABSTRACT: 
      <br><br></a_abstract>
     
    G. M. Verkhivker.
    <a href="http://onlinelibrary.wiley.com/doi/10.1002/prot.20351/abstract">Protein conformational transitions  coupled to binding in molecular recognition of unstructured proteins: Deciphering the role of intermolecular  interactions in computational structure prediction of the p27 protein bound to the cyclin A-cyclin-dependent kinase 2 complex.</a>
    Proteins: Struct. Funct. Bionformatics 58:706-716, 2005.
    <a href= "https://www.ncbi.nlm.nih.gov/pubmed/15609350"> PUBMED. </a>
    <a href="#abstractProt" id="l25">Abstract.</a>
    <br><br>
      <a_abstract NAME="abstractProt" id="t25"> 
      ABSTRACT: The relationship between folding mechanism coupled to binding and structure prediction of the tertiary complexes is studied for the p27Kip1 protein which has an intrinsically disordered unbound form and undergoes a functional folding transition during complex formation with the phosphorylated cyclin A–cyclin-dependent kinase 2 (Cdk2) binary complex. Hierarchy of p27Kip1 structural loss determined in our earlier studies from temperature–induced Monte Carlo simulations and subsequent characterization of the transition state ensemble (TSE) for the folding reaction have shown that simultaneous ordering of the p27Kip1 native intermolecular interface for the β-hairpin and β-strand secondary structure elements is critical for nucleating a rapid kinetic transition to the native tertiary complex. In the present study, we investigate the effect of forming specific intermolecular interactions on structure prediction of the p27Kip1 tertiary complex. By constraining different secondary structure elements of p27Kip1 in their native bound conformations and conducting multiple simulated annealing simulations, we analyze differences in the success rate of predicting the native structure of p27Kip1 in the tertiary complex. In accordance with the nucleation–condensation mechanism, we have found that further stabilization of the native intermolecular interface for the β-hairpin and β-strand elements of p27Kip1, that become ordered in the TSE, but are hardly populated in the unbound state, results in a consistent acquisition of the native bound structure. Conversely, the excessive stablization of the local secondary structure elements, which are rarely detected in the TSE, has a detrimental effect on convergence to the native bound structure. 
      <br><br></a_abstract>

   
   <p class="sub_ctrXS">More Publications</p> 
    
     <a href="/Publications/2015o2019.php">2015 - 2019 (14 Publications) </a><br>
     <a href="/Publications/2010o2014.php">2010 - 2014 (17 Publications)</a><br>
     <a href="/Publications/2000o2004.php">2000 - 2004 (17 Publications) </a><br>
     <a href="/Publications/1995o1999.php">1995 - 1999 (21 Publications) </a><br>
     <a href="/Publications/1990o1994.php">1990 - 1994 (9 Publications)</a><br>
     <a href="/Publications/o1990.php">Before 1990     (15 Publications) </a><br><br>
   <!-- InstanceEndEditable -->
      <div class="clear"></div>
    
    
    
    </div>
  </div>
  <footer >
    <div id="bottom">
      <div class="clear"></div>
    </div>
  </footer> <div id="subfooter">©2016 Chapman University • One University Drive, Orange, CA 92866 • Phone: (714) 997-6815</div>
</div>
</body>
<!-- InstanceEnd --></html>