<!DOCTYPE html>
<html lang="en"><!-- InstanceBegin template="/Templates/subpage.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<!-- InstanceBeginEditable name="meta" -->
<!-- InstanceEndEditable -->
<!--[if gte IE 9]><meta http-equiv="X-UA-Compatible" content="IE=Edge"><![endif]-->
<!-- InstanceBeginEditable name="doctitle" -->
<title>Publications 2015 - Present | Verkhivker Research Group</title>
<!-- InstanceEndEditable -->
<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" type="text/css" href="../css/harv.css">
<link rel="stylesheet" type="text/css" href="../css/harvdrop.css">
<!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><script src="/js/html5shiv.js"></script><![endif]-->
<script src='http://code.jquery.com/jquery-latest.min.js' type='text/javascript'></script><script src="/js/jquery.ui.totop.js"></script>
<!-- InstanceBeginEditable name="head" --><style>
.text_box, p {
	text-align: left !important;
}
#middle #mainsub .biblio-abstract-link.toggle {
}
</style>

<!-- InstanceEndEditable -->
</head>
<body>
<div id="container">
  <div id="smNavBk"><a href="http://compbiosciences.chapman.edu/index.php"><img src="../images/header400light.png" width="332" height="58" alt="Chapman University" style="float: left; margin-left: 10px; margin-top: 10px;"></a>
    <div id="collegeLinks" style="font-size: 24px;">
      <p><a href="https://www.chapman.edu/scst/"><strong><img src="../images/xdz1wyqhwtmxxekidsyy.jpg" width="35" height="35" alt=""/>Schmid College of Science and Technology</strong></a></p>
      <p><a href="http://www.chapman.edu/"><img src="../images/open-graph-cu-logo.png" alt="Chapman University" width="35" height="30" title="Chapman University"/> <span style="font-style: normal; font-size: 18pt;">Chapman University</span></a></p>
    </div>
    <div class="clear"></div>
  </div>
  </header>
  <nav id="topMenu" class="desktop"><!-- #BeginLibraryItem "/Library/topMenu.lbi" --> <nav id="topMenu" class="desktop"><ul id="TopNav" class="MenuBarHorizontal">
      <li> <a href="../index.php">Home</a></li>
      <li><a href="../members.php">Members</a>          
  <li><a href="../Research/research.php" class="MenuBarItemSubmenu">Research</a>
        <ul>
          <li><a href="../Research/computationalCancerBiology.php" >Computational Biology and Translational Bioinformatics in Cancer Research</a></li>
          <li><a href="../Research/proteinKinasesResearch.php" >Protein Kinases</a></li>
          <li><a href="../Research/molecularChaperonesAndCancerResearch.php">Computatoinal Studies of Molecular Chaperones</a></li>
          <li><a href="../Research/translationalBioinformatics.php" >Translational Bioinformatics and Drug Discovery</a></li>
          <li><a href="../Research/relatedResearch.php">Related Links</a></li>
        </ul>
  </li>
      <li><a href="publications.php" class="MenuBarItemSubmenu">Publications</a>          
        <ul>
          <li><a href="2015o2019.php">2015-2019</a></li>
          <li><a href="2010o2014.php">2010-2014</a></li>
          <li><a href="2005o2009.php">2005-2009</a></li>
          <li><a href="2000o2004.php">2000-2004</a></li>
          <li><a href="1995o1999.php">1995-1999</a></li>
          <li><a href="1990o1994.php">1990-1994</a></li>
          <li><a href="o1990.php">Before 1990</a></li>
        </ul>
    <li><a href="../aboutUs.php">About Us</a>  </li>
    <li><a href="../contact.php">Contact Us</a></li>
    </ul>
    <div class="clear"></div></nav><!-- #EndLibraryItem --></nav></div>
  <div id="middle">
    <div id="mainsub"><!-- InstanceBeginEditable name="crumbs" --><script>breadcrumb()</script><!-- InstanceEndEditable -->
     
     
     
    <div class="clear"></div>
    
    
   <!-- InstanceBeginEditable name="main_content_center" -->
      <p class="header">Publications: 2015-2019</p>
      <p class="sub_ctrXS">2017</p>
  
       
        G. Stetz G, A. Tse A, J. Bouz, GM Verkhivker. Allosteric Mechanisms of BRAF Kinases Regulation and Combating Drug Resistance of Kinase Inhibitors: A Synergistic Perspective from Computational Systems Biology and Biophysical Studies. J. Mol. Biol. 2017, in press. <br> <br>
        
        
      	N. Lawless, E. Berrigan, K. Blacklock, G. Verkhivker. Computational modeling of Hsp90 molecular chaperone structure, dynamics   and function: a synergistic perspective from biophysical simulations and systems biology analysis. Molecular Systems Biology,  2017, in press. <br><br>
      	
      	GM Verkhivker, K. Blacklock, G. Stetz,  Computational modeling of Hsp90 molecular interactions Methods in Molecular Biology, Molecular Chaperones edition. Invited Chapter, 2017, in press.<br><br>
       
        Stetz G, Verkhivker GM. Computational Analysis of Residue Interaction Networks and Coevolutionary Relationships in the Hsp70 Chaperones: A Community-Hopping Model of Allosteric Regulation and Communication. Plos Comput. Biol. 2017, in press.<br><br>
        
      <p class="sub_ctrXS">2016</p>    
         
	  Tiwari RK, Brown A, Sadeghiani N, Shirazi AN, Bolton J, Tse A, Verkhivker G, Parang K, Sun G. <a href="http://onlinelibrary.wiley.com/doi/10.1002/cmdc.201600387/abstract;jsessionid=E8D8602B401D0E717A09534D41EC8780.f03t03"> Design, Synthesis, and Evaluation of Dasatinib-Amino Acid and Dasatinib-Fatty Acid Conjugates as Protein Tyrosine Kinase Inhibitors. </a> ChemMedChem, 2016 Nov 22.<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1002%2Fcmdc.201600387"> PUBMED.</a><a href="#abstractDesign" id="btn1"> Abstract.</a><br><br>
       
       
       <a_abstract NAME="abstractDesign" id="x1">ABSTRACT: Derivatives of the tyrosine kinase inhibitor dasatinib were synthesized by esterification with 25 carboxylic acids, including amino acids and fatty acids, thereby extending the drug to interact with more diverse sites and to improve specificity. The dasatinib-l-arginine derivative (Das-R, 7) was found to be the most potent of the inhibitors tested, with IC50 values of 4.4, <0.25, and <0.45 nm against Csk, Src, and Abl kinases, respectively. The highest selectivity ratio obtained in our study, 91.4 Csk/Src, belonged to compound 18 (Das-C10 ) with an IC50 value of 3.2 μm for Csk compared with 35 nm for Src. Furthermore, many compounds displayed increased selectivity toward Src over Abl. Compounds 15 (Das-glutamic acid) and 13 (Das-cysteine) demonstrated the largest gains (10.2 and 10.3 Abl/Src IC50 ratios). Das-R (IC50 =2.06 μm) was significantly more potent than the parent dasatinib (IC50 =26.3 μm) against Panc-1 cells, whereas both compounds showed IC50 <51.2 pm against BV-173 and K562 cells. Molecular modeling and binding free energy simulations revealed good agreements with the experimental results and rationalized the differences in selectivity among the studied compounds. Integration of experimental and computational approaches in the design and biochemical screening of dasatinib derivatives facilitated rational engineering and diversification of the dasatinib scaffold, providing useful insight into mechanisms of kinase selectivity.<br><br></a_abstract>
        
  
	  Verkhivker GM.<a href="http://www.eurekaselect.com/146105/article"> Leveraging Structural Diversity and Allosteric Regulatory Mechanisms of Protein Kinases in the Discovery of Small Molecule Inhibitors.</a> Curr Med Chem. 2016 Oct 6. [Epub ahead of print]<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Leveraging+Structural+Diversity+and+Allosteric+Regulatory+Mechanisms+of+Protein+Kinases+in+the+Discovery+of+Small+Molecule+Inhibitors"> PUBMED.</a><a href="#abstractLeverage" id="btn2"> Abstract.</a><br><br>
     	
     	<a_abstract NAME="abstractExplo" id="x2">ABSTRACT: Protein kinases are versatile molecule switches that govern functional processes in signal transduction networks and regulate fundamental biological processes of cell cycle and organism development. The continuous growth of biological information and a remarkable breath of structural, genetic, and pharmacological studies on protein kinase genes have significantly advanced our knowledge of the kinase activation, drug binding and allosteric mechanisms underlying kinase regulation and interactions in signaling cascades. . Structural and biochemical studies of the genetic and molecular determinants of protein kinases binding with inhibitors have been the cornerstone of drug discovery efforts in clinical oncology leading to proliferation of effective anticancer therapies. Recent advances in understanding allosteric regulation of protein kinases have fueled unprecedented efforts aiming in the discovery of targeted and allosteric kinase inhibitors that can combat cancer mutants and are at the forefront of the precision medicine initiative in oncology. Despite diversity of regulatory scenarios underlying kinase functions, dimerization-driven activation is a common mechanism of allosteric regulation that is shared by many protein kinase families, most notably ErbB and BRAF kinases that play a central role in growth factor signaling and human disease. In this review, we focused on structural, biochemical and computational studies of the ErbB and BRAF kinases and discuss how diversity of the structural landscape for these kinase genes and dimerization-dependent mechanisms of their regulation can be leveraged in the design and discovery of kinase inhibitors and allosteric modulators of kinase activation. The lessons from this analysis could inform discovery of specific targeted therapies and robust drug combinations for cancer treatment.<br><br></a_abstract>
      
	  Tse A, Verkhivker GM. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0166583"> Exploring Molecular Mechanisms of Paradoxical Activation in the BRAF Kinase Dimers: Atomistic Simulations of Conformational Dynamics and Modeling of Allosteric Communication Networks and Signaling Pathways.</a> Plos ONE, 2016, Nov 18;11(11):e0166583.<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Exploring+Molecular+Mechanisms+of+Paradoxical+Activation+in+the+BRAF+Kinase+Dimers%3A+Atomistic+Simulations+of+Conformational+Dynamics+and+Modeling+of+Allosteric+Communication+Networks+and+Signaling+Pathways"> PUBMED.</a><a href="#abstractExplo" id="btn3"> Abstract.</a><br><br>
        
        <a_abstract NAME="abstractExplo" id="x3"> ABSTRACT: The recent studies have revealed that most BRAF inhibitors can paradoxically induce kinase activation by promoting dimerization and enzyme transactivation. Despite rapidly growing number of structural and functional studies about the BRAF dimer complexes, the molecular basis of paradoxical activation phenomenon is poorly understood and remains largely hypothetical. In this work, we have explored the relationships between inhibitor binding, protein dynamics and allosteric signaling in the BRAF dimers using a network-centric approach. Using this theoretical framework, we have combined molecular dynamics simulations with coevolutionary analysis and modeling of the residue interaction networks to determine molecular determinants of paradoxical activation. We have investigated functional effects produced by paradox inducer inhibitors PLX4720, Dabrafenib, Vemurafenib and a paradox breaker inhibitor PLX7904. Functional dynamics and binding free energy analyses of the BRAF dimer complexes have suggested that negative cooperativity effect and dimer-promoting potential of the inhibitors could be important drivers of paradoxical activation. We have introduced a protein structure network model in which coevolutionary residue dependencies and dynamic maps of residue correlations are integrated in the construction and analysis of the residue interaction networks. The results have shown that coevolutionary residues in the BRAF structures could assemble into independent structural modules and form a global interaction network that may promote dimerization. We have also found that BRAF inhibitors could modulate centrality and communication propensities of global mediating centers in the residue interaction networks. By simulating allosteric communication pathways in the BRAF structures, we have determined that paradox inducer and breaker inhibitors may activate specific signaling routes that correlate with the extent of paradoxical activation. While paradox inducer inhibitors may facilitate a rapid and efficient communication via an optimal single pathway, the paradox breaker may induce a broader ensemble of suboptimal and less efficient communication routes. The central finding of our study is that paradox breaker PLX7904 could mimic structural, dynamic and network features of the inactive BRAF-WT monomer that may be required for evading paradoxical activation. The results of this study rationalize the existing structure-functional experiments by offering a network-centric rationale of the paradoxical activation phenomenon. We argue that BRAF inhibitors that amplify dynamic features of the inactive BRAF-WT monomer and intervene with the allosteric interaction networks may serve as effective paradox breakers in cellular environment.<br><br></a_abstract>
      
      
	  Verkhivker GM. <a href="http://dx.doi.org/10.1039/c6mb00298f"> Molecular dynamics simulations and modelling of the residue interaction networks in the BRAF kinase complexes with small molecule inhibitors: probing the allosteric effects of ligand-induced kinase dimerization and paradoxical activation.</a> Mol Biosyst. 2016 Oct 20;12(10):3146-65. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1039%2Fc6mb00298f"> PUBMED.</a> <a href="#abstractMolDyn" id="btn4">Abstract.</a><br><br>
      	
      	<a_abstract NAME="abstractMolDyn" id="x4"> ABSTRACT: Protein kinases are central to proper functioning of cellular networks and are an integral part of many signal transduction pathways. The family of protein kinases represents by far the largest and most important class of therapeutic targets in oncology. Dimerization-induced activation has emerged as a common mechanism of allosteric regulation in BRAF kinases, which play an important role in growth factor signalling and human diseases. Recent studies have revealed that most of the BRAF inhibitors can induce dimerization and paradoxically stimulate enzyme transactivation by conferring an active conformation in the second monomer of the kinase dimer. The emerging connections between inhibitor binding and BRAF kinase domain dimerization have suggested a molecular basis of the activation mechanism in which BRAF inhibitors may allosterically modulate the stability of the dimerization interface and affect the organization of residue interaction networks in BRAF kinase dimers. In this work, we integrated structural bioinformatics analysis, molecular dynamics and binding free energy simulations with the protein structure network analysis of the BRAF crystal structures to determine dynamic signatures of BRAF conformations in complexes with different types of inhibitors and probe the mechanisms of the inhibitor-induced dimerization and paradoxical activation. The results of this study highlight previously unexplored relationships between types of BRAF inhibitors, inhibitor-induced changes in the residue interaction networks and allosteric modulation of the kinase activity. This study suggests a mechanism by which BRAF inhibitors could promote or interfere with the paradoxical activation of BRAF kinases, which may be useful in informing discovery efforts to minimize the unanticipated adverse biological consequences of these therapeutic agents.<br><br></a_abstract> 
    
    
	  Stetz G, Verkhivker GM. <a href="http://pubs.acs.org/doi/abs/10.1021/acs.jcim.5b00755"> Probing Allosteric Inhibition Mechanisms of the Hsp70 Chaperone Proteins Using Molecular Dynamics Simulations and Analysis of the Residue Interaction Networks.</a> J Chem Inf Model. 2016 Aug 22;56(8):1490-517. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Probing+Allosteric+Inhibition+Mechanisms+of+the+Hsp70+Chaperone+Proteins+Using+Molecular+Dynamics+Simulations+and+Analysis+of+the+Residue+Interaction+Networks"> PUBMED. </a> <a href="#abstractProb" id="btn5">Abstract.</a><br><br>
         
        <a_abstract NAME="abstractProb" id="x5"> ABSTRACT:  Although molecular mechanisms of allosteric regulation in the Hsp70 chaperones have been extensively studied at both structural and functional levels, the current understanding of allosteric inhibition of chaperone activities by small molecules is still lacking. In the current study, using a battery of computational approaches, we probed allosteric inhibition mechanisms of E. coli Hsp70 (DnaK) and human Hsp70 proteins by small molecule inhibitors PET-16 and novolactone. Molecular dynamics simulations and binding free energy analysis were combined with network-based modeling of residue interactions and allosteric communications to systematically characterize and compare molecular signatures of the apo form, substrate-bound, and inhibitor-bound chaperone complexes. The results suggested a mechanism by which the allosteric inhibitors may leverage binding energy hotspots in the interaction networks to stabilize a specific conformational state and impair the interdomain allosteric control. Using the network-based centrality analysis and community detection, we demonstrated that substrate binding may strengthen the connectivity of local interaction communities, leading to a dense interaction network that can promote an efficient allosteric communication. In contrast, binding of PET-16 to DnaK may induce significant dynamic changes and lead to a fractured interaction network and impaired allosteric communications in the DnaK complex. By using a mechanistic-based analysis of distance fluctuation maps and allosteric propensities of protein residues, we determined that the allosteric network in the PET-16 complex may be small and localized due to the reduced communication and low cooperativity of the substrate binding loops, which may promote the higher rates of substrate dissociation and the decreased substrate affinity. In comparison with the significant effect of PET-16, binding of novolactone to HSPA1A may cause only moderate network changes and preserve allosteric coupling between the allosteric pocket and the substrate binding region. The impact of novolactone on the conformational dynamics and allosteric communications in the HSPA1A complex was comparable to the substrate effect, which is consistent with the experimental evidence that PET-16, but not novolactone binding, can significantly decrease substrate affinity. We argue that the unique dynamic and network signatures of PET-16 and novolactone may be linked with the experimentally observed functional effects of these inhibitors on allosteric regulation and substrate binding.<br><br></a_abstract> 
          
    
    	Verkhivker GM.<a href="http://www.worldscientific.com/doi/abs/10.1142/9789814749411_0005">Integrating genetic and structural data on human protein kinome in network-based modeling of kinase sensitivities and resistance to targeted and personalized anticancer drugs.  </a>Pac Symp Biocomput. 2016; 21:45-56. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26776172"> PUBMED. </a><a href="#abstractIntegrat" id="btn6">Abstract.</a><br><br>
    	
    	<a_abstract NAME="abstractIntegrat" id="x6"> ABSTRACT: The human protein kinome presents one of the largest protein families that orchestrate functional processes in complex cellular networks, and when perturbed, can cause various cancers. The abundance and diversity of genetic, structural, and biochemical data underlies the complexity of mechanisms by which targeted and personalized drugs can combat mutational profiles in protein kinases. Coupled with the evolution of system biology approaches, genomic and proteomic technologies are rapidly identifying and charactering novel resistance mechanisms with the goal to inform rationale design of personalized kinase drugs. Integration of experimental and computational approaches can help to bring these data into a unified conceptual framework and develop robust models for predicting the clinical drug resistance. In the current study, we employ a battery of synergistic computational approaches that integrate genetic, evolutionary, biochemical, and structural data to characterize the effect of cancer mutations in protein kinases. We provide a detailed structural classification and analysis of genetic signatures associated with oncogenic mutations. By integrating genetic and structural data, we employ network modeling to dissect mechanisms of kinase drug sensitivities to oncogenic EGFR mutations. Using biophysical simulations and analysis of protein structure networks, we show that conformational-specific drug binding of Lapatinib may elicit resistant mutations in the EGFR kinase that are linked with the ligand-mediated changes in the residue interaction networks and global network properties of key residues that are responsible for structural stability of specific functional states. A strong network dependency on high centrality residues in the conformation-specific Lapatinib-EGFR complex may explain vulnerability of drug binding to a broad spectrum of mutations and the emergence of drug resistance. Our study offers a systems-based perspective on drug design by unravelling complex relationships between robustness of targeted kinase genes and binding specificity of targeted kinase drugs. We discuss how these approaches can exploit advances in chemical biology and network science to develop novel strategies for rationally tailored and robust personalized drug therapies.<br><br></a_abstract>
    	
    
    
    
   <p class="sub_ctrXS">2015</p> 

    
	  Stetz G, Verkhivker GM.  <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0143752"> Dancing through Life: Molecular Dynamics Simulations and Network-Centric Modeling of Allosteric Mechanisms in Hsp70 and Hsp110 Chaperone Proteins.</a>  PLoS One. 2015 Nov 30;10(11):e0143752. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1371%2Fjournal.pone.0143752"> PUBMED. </a><a href="#abstractDance" id="btn7">Abstract.</a><br><br>
      
      <a_abstract NAME="abstractDance" id="x7"> ABSTRACT: Hsp70 and Hsp110 chaperones play an important role in regulating cellular processes that involve protein folding and stabilization, which are essential for the integrity of signaling networks. Although many aspects of allosteric regulatory mechanisms in Hsp70 and Hsp110 chaperones have been extensively studied and significantly advanced in recent experimental studies, the atomistic picture of signal propagation and energetics of dynamics-based communication still remain unresolved. In this work, we have combined molecular dynamics simulations and protein stability analysis of the chaperone structures with the network modeling of residue interaction networks to characterize molecular determinants of allosteric mechanisms. We have shown that allosteric mechanisms of Hsp70 and Hsp110 chaperones may be primarily determined by nucleotide-induced redistribution of local conformational ensembles in the inter-domain regions and the substrate binding domain. Conformational dynamics and energetics of the peptide substrate binding with the Hsp70 structures has been analyzed using free energy calculations, revealing allosteric hotspots that control negative cooperativity between regulatory sites. The results have indicated that cooperative interactions may promote a population-shift mechanism in Hsp70, in which functional residues are organized in a broad and robust allosteric network that can link the nucleotide-binding site and the substrate-binding regions. A smaller allosteric network in Hsp110 structures may elicit an entropy-driven allostery that occurs in the absence of global structural changes. We have found that global mediating residues with high network centrality may be organized in stable local communities that are indispensable for structural stability and efficient allosteric communications. The network-centric analysis of allosteric interactions has also established that centrality of functional residues could correlate with their sensitivity to mutations across diverse chaperone functions. This study reconciles a wide spectrum of structural and functional experiments by demonstrating how integration of molecular simulations and network-centric modeling may explain thermodynamic and mechanistic aspects of allosteric regulation in chaperones.<br><br></a_abstract>
       
	  A.Tse, K. GM. Verkhivker. <a href="http://pubs.acs.org/doi/10.1021/acs.jcim.5b00240"> Molecular Dynamics Simulations and Structural Network Analysis of c-Abl and c-Src Kinase Core Proteins: Capturing Allosteric Mechanisms and Communication Pathways from Residue Centrality.</a> Tse A, Verkhivker GM. J Chem Inf Model. 2015 Aug 24;55(8):1645-62. Epub 2015 Aug 12. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26236953"> PUBMED. </a><a href="#abstractStructNet" id="btn8">Abstract.</a><br><br>
    
    <a_abstract NAME="abstractStructNet" id="x8"> ABSTRACT: The Abl and Src tyrosine kinases play a fundamental regulatory role in orchestrating functional processes in cellular networks and represent an important class of therapeutic targets. Crystallographic studies of these kinases have revealed a similar structural organization of multidomain complexes that confers salient features of their regulatory mechanisms. Molecular characterization of the interaction networks and regulatory residues by which the SH3 and SH2 domains act cooperatively with the catalytic domain to suppress or promote kinase activation presents an active area of structural, biochemical, and computational investigations. In this work, we combine biophysical simulations with computational modeling of the residue interaction networks to characterize allosteric mechanisms of kinase regulation and gain insight into differential sensitivity of c-Abl and c-Src kinases to specific drug binding. Using these approaches, we examine dynamics of cooperative rearrangements in the residue interaction networks and elucidate the structural role of regulatory residues responsible for modulation of kinase activity. We have found that global network parameters such as residue centrality can unambiguously distinguish functional sites that are responsible for mediating allosteric interactions in the regulatory assemblies. This study has revealed mechanistic aspects of allosteric mechanisms and communication pathways by which the SH3 and SH2 domains may exert their regulatory influence on the catalytic domain and kinase activity. We have also found that high centrality residues can be linked to each other to form efficient and robust routes that transmit allosteric signals between spatially separated regulatory regions. The presented results have demonstrated that global features of the residue interaction networks may serve as transparent and robust indicators of kinase regulatory mechanisms and accurately pinpoint key functional residues.<br><br></a_abstract>
     
	  A.Tse, K. GM. Verkhivker.<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2015/MB/C5MB00246J#!divAbstract"> Small-world networks of residue interactions in the Abl kinase complexes with cancer drugs: topology of allosteric communication pathways can determine drug resistance effects. </a> Mol Biosyst. 2015 Jul;11(7):2082-95. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25996455"> PUBMED. </a><a href="#abstractSmallWorld" id="btn9">Abstract.</a><br><br>
       
     <a_abstract NAME="abstractSmallWorld" id="x9"> ABSTRACT: The human protein kinases play a fundamental regulatory role in orchestrating functional processes in complex cellular networks. Understanding how conformational equilibrium between functional kinase states can be modulated by ligand binding or mutations is critical for quantifying molecular basis of allosteric regulation and drug resistance. In this work, molecular dynamics simulations of the Abl kinase complexes with cancer drugs (Imatinib and Dasatinib) were combined with structure-based network modeling to characterize dynamics of the residue interaction networks in these systems. The results have demonstrated that structural architecture of kinase complexes can produce a small-world topology of the interaction networks. Our data have indicated that specific Imatinib binding to a small number of highly connected residues could lead to network-bridging effects and allow for efficient allosteric communication, which is mediated by a dominant pathway sensitive to the unphosphorylated Abl state. In contrast, Dasatinib binding to the active kinase form may activate a broader ensemble of allosteric pathways that are less dependent on the phosphorylation status of Abl and provide a better balance between the efficiency and resilience of signaling routes. Our results have unveiled how differences in the residue interaction networks and allosteric communications of the Abl kinase complexes can be directly related to drug resistance effects. This study offers a plausible perspective on how efficiency and robustness of the residue interaction networks and allosteric pathways in kinase structures may be associated with protein responses to drug binding.<br><br></a_abstract>
        
	  A.Tse, K. GM. Verkhivker. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0130203"> Molecular Determinants Underlying Binding Specificities of the ABL Kinase Inhibitors: Combining Alanine Scanning of Binding Hot Spots with Network Analysis of Residue Interactions and Coevolution.</a> PLoS One. 2015 Jun 15;10(6):e0130203. eCollection 2015. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26075886"> PUBMED. </a><a href="#abstractMolecDet" id="btn10">Abstract.</a><br><br> 
       
     <a_abstract NAME="abstractMolecDet" id="x10"> ABSTRACT: Quantifying binding specificity and drug resistance of protein kinase inhibitors is of fundamental importance and remains highly challenging due to complex interplay of structural and thermodynamic factors. In this work, molecular simulations and computational alanine scanning are combined with the network-based approaches to characterize molecular determinants underlying binding specificities of the ABL kinase inhibitors. The proposed theoretical framework unveiled a relationship between ligand binding and inhibitor-mediated changes in the residue interaction networks. By using topological parameters, we have described the organization of the residue interaction networks and networks of coevolving residues in the ABL kinase structures. This analysis has shown that functionally critical regulatory residues can simultaneously embody strong coevolutionary signal and high network centrality with a propensity to be energetic hot spots for drug binding. We have found that selective (Nilotinib) and promiscuous (Bosutinib, Dasatinib) kinase inhibitors can use their energetic hot spots to differentially modulate stability of the residue interaction networks, thus inhibiting or promoting conformational equilibrium between inactive and active states. According to our results, Nilotinib binding may induce a significant network-bridging effect and enhance centrality of the hot spot residues that stabilize structural environment favored by the specific kinase form. In contrast, Bosutinib and Dasatinib can incur modest changes in the residue interaction network in which ligand binding is primarily coupled only with the identity of the gate-keeper residue. These factors may promote structural adaptability of the active kinase states in binding with these promiscuous inhibitors. Our results have related ligand-induced changes in the residue interaction networks with drug resistance effects, showing that network robustness may be compromised by targeted mutations of key mediating residues. This study has outlined mechanisms by which inhibitor binding could modulate resilience and efficiency of allosteric interactions in the kinase structures, while preserving structural topology required for catalytic activity and regulation.<br><br></a_abstract>
     
     
     <p class="sub_ctrXS">More Publications</p> 
     
     <a href="/Publications/2010o2014.php">2010 - 2014 (17 Publications)</a><br>
     <a href="/Publications/2005o2009.php">2005 - 2009 (26 Publications) </a><br>
     <a href="/Publications/2000o2004.php">2000 - 2004 (17 Publications) </a><br>
     <a href="/Publications/1995o1999.php">1995 - 1999 (21 Publications) </a><br>
     <a href="/Publications/1990o1994.php">1990 - 1994 (9 Publications)</a><br>
     <a href="/Publications/o1990.php">Before 1990     (15 Publications) </a><br><br>
   <!-- InstanceEndEditable -->
      <div class="clear"></div>
    
    
    
    </div>
  </div>
  <footer >
    <div id="bottom">
      <div class="clear"></div>
    </div>
  </footer> <div id="subfooter">©2016 Chapman University • One University Drive, Orange, CA 92866 • Phone: (714) 997-6815</div>
</div>
</body>
<!-- InstanceEnd --></html>