<!DOCTYPE html>
<html lang="en"><!-- InstanceBegin template="/Templates/subpage.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<!-- InstanceBeginEditable name="meta" -->
<!-- InstanceEndEditable -->
<!--[if gte IE 9]><meta http-equiv="X-UA-Compatible" content="IE=Edge"><![endif]-->
<!-- InstanceBeginEditable name="doctitle" -->
<title>Publications | Verkhivker Research Group</title>
<!-- InstanceEndEditable -->
<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" type="text/css" href="../css/harv.css">
<link rel="stylesheet" type="text/css" href="../css/harvdrop.css">
<!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><script src="/js/html5shiv.js"></script><![endif]-->
<script src='http://code.jquery.com/jquery-latest.min.js' type='text/javascript'></script><script src="/js/jquery.ui.totop.js"></script>
<!-- InstanceBeginEditable name="head" --><style>
.text_box, p {
	text-align: left !important;
}
	
	
	
#middle #mainsub .biblio-abstract-link.toggle {
}
</style>
<!-- InstanceEndEditable -->
</head>
<body>
<div id="container">
  <div id="smNavBk"><a href="http://compbiosciences.chapman.edu/index.php"><img src="../images/header400light.png" width="332" height="58" alt="Chapman University" style="float: left; margin-left: 10px; margin-top: 10px;"></a>
    <div id="collegeLinks" style="font-size: 24px;">
      <p><a href="https://www.chapman.edu/scst/"><strong><img src="../images/xdz1wyqhwtmxxekidsyy.jpg" width="35" height="35" alt=""/>Schmid College of Science and Technology</strong></a></p>
      <p><a href="http://www.chapman.edu/"><img src="../images/open-graph-cu-logo.png" alt="Chapman University" width="35" height="30" title="Chapman University"/> <span style="font-style: normal; font-size: 18pt;">Chapman University</span></a></p>
    </div>
    <div class="clear"></div>
  </div>
  </header>
  <nav id="topMenu" class="desktop"><!-- #BeginLibraryItem "/Library/topMenu.lbi" --> <nav id="topMenu" class="desktop"><ul id="TopNav" class="MenuBarHorizontal">
      <li> <a href="../index.php">Home</a></li>
      <li><a href="../members.php">Members</a>          
  <li><a href="../Research/research.php" class="MenuBarItemSubmenu">Research</a>
        <ul>
          <li><a href="../Research/computationalCancerBiology.php" >Computational Biology and Translational Bioinformatics in Cancer Research</a></li>
          <li><a href="../Research/proteinKinasesResearch.php" >Protein Kinases</a></li>
          <li><a href="../Research/molecularChaperonesAndCancerResearch.php">Computatoinal Studies of Molecular Chaperones</a></li>
          <li><a href="../Research/translationalBioinformatics.php" >Translational Bioinformatics and Drug Discovery</a></li>
          <li><a href="../Research/relatedResearch.php">Related Links</a></li>
        </ul>
  </li>
      <li><a href="publications.php" class="MenuBarItemSubmenu">Publications</a>          
        <ul>
          <li><a href="2015o2019.php">2015-2019</a></li>
          <li><a href="2010o2014.php">2010-2014</a></li>
          <li><a href="2005o2009.php">2005-2009</a></li>
          <li><a href="2000o2004.php">2000-2004</a></li>
          <li><a href="1995o1999.php">1995-1999</a></li>
          <li><a href="1990o1994.php">1990-1994</a></li>
          <li><a href="o1990.php">Before 1990</a></li>
        </ul>
    <li><a href="../aboutUs.php">About Us</a>  </li>
    <li><a href="../contact.php">Contact Us</a></li>
    </ul>
    <div class="clear"></div></nav><!-- #EndLibraryItem --></nav></div>
  <div id="middle">
    <div id="mainsub"><!-- InstanceBeginEditable name="crumbs" --><script>breadcrumb()</script><!-- InstanceEndEditable -->
     
     
     
    <div class="clear"></div>
    
    
   <!-- InstanceBeginEditable name="main_content_center" -->
      <p class="header">Publications</p>
      <A HREF="#2017"> 2017 |</A>
      <A HREF="#2016"> 2016 |</A>
      <A HREF="#2015"> 2015 |</A>
      <A HREF="#2014"> 2014 |</A>
      <A HREF="#2013"> 2013 |</A>
      <A HREF="#2012"> 2012 |</A>
      <A HREF="#2011"> 2010 - 2011 |</A>
      <A HREF="#B2010">Before 2010 </A>

        <br><br>
      </p>
      <A NAME="2017"></A>
    <p class="sub_ctrXS">2017</p>
  
       
        Stetz G, Tse A, Bouz J, Verkhivker GM. Allosteric Mechanisms of BRAF Kinases Regulation and Combating Drug Resistance of Kinase Inhibitors: A Synergistic Perspective from Computational Systems Biology and Biophysical Studies. <a_journal_title> J. Mol. Biol. </a_journal_title> 2017, in press. <br> <br>
        
        
      	Lawless N, Berrigan E, Blacklock K, Verkhivker GM. Computational modeling of Hsp90 molecular chaperone structure, dynamics   and function: a synergistic perspective from biophysical simulations and systems biology analysis. Molecular Systems Biology.  2017, in press. <br><br>
      	
      	Verkhivker GM, Blacklock K, Stetz G.  Computational modeling of Hsp90 molecular interactions Methods in Molecular Biology, Molecular Chaperones edition. Invited Chapter, 2017, in press.<br><br>
       
        Stetz G, Verkhivker GM. Computational Analysis of Residue Interaction Networks and Coevolutionary Relationships in the Hsp70 Chaperones: A Community-Hopping Model of Allosteric Regulation and Communication. Plos Comput. Biol. 2017, in press.<br><br>
        
      <A NAME="2016"></A> 
      <p class="sub_ctrXS">2016</p>    
         
	  Tiwari RK, Brown A, Sadeghiani N, Shirazi AN, Bolton J, Tse A, Verkhivker GM, Parang K, Sun G. <a href="http://onlinelibrary.wiley.com/doi/10.1002/cmdc.201600387/abstract;jsessionid=E8D8602B401D0E717A09534D41EC8780.f03t03"> Design, Synthesis, and Evaluation of Dasatinib-Amino Acid and Dasatinib-Fatty Acid Conjugates as Protein Tyrosine Kinase Inhibitors. </a> <a_journal_title>  ChemMedChem </a_journal_title>, 2016 Nov 22.<a_PUBMED href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1002%2Fcmdc.201600387"> PUBMED.</a_PUBMED><a href="#abstractDesign" id="btn1"> Abstract.</a><br><br>
       
       
	  <a_abstract NAME="abstractDesign" id="x1" > <form style="border: 1px solid; height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> Derivatives of the tyrosine kinase inhibitor dasatinib were synthesized by esterification with 25 carboxylic acids, including amino acids and fatty acids, thereby extending the drug to interact with more diverse sites and to improve specificity. The dasatinib-l-arginine derivative (Das-R, 7) was found to be the most potent of the inhibitors tested, with IC50 values of 4.4, <0.25, and <0.45 nm against Csk, Src, and Abl kinases, respectively. The highest selectivity ratio obtained in our study, 91.4 Csk/Src, belonged to compound 18 (Das-C10 ) with an IC50 value of 3.2 μm for Csk compared with 35 nm for Src. Furthermore, many compounds displayed increased selectivity toward Src over Abl. Compounds 15 (Das-glutamic acid) and 13 (Das-cysteine) demonstrated the largest gains (10.2 and 10.3 Abl/Src IC50 ratios). Das-R (IC50 =2.06 μm) was significantly more potent than the parent dasatinib (IC50 =26.3 μm) against Panc-1 cells, whereas both compounds showed IC50 <51.2 pm against BV-173 and K562 cells. Molecular modeling and binding free energy simulations revealed good agreements with the experimental results and rationalized the differences in selectivity among the studied compounds. Integration of experimental and computational approaches in the design and biochemical screening of dasatinib derivatives facilitated rational engineering and diversification of the dasatinib scaffold, providing useful insight into mechanisms of kinase selectivity.</form><br><br></a_abstract>
        
  
	  Verkhivker GM.<a href="http://www.eurekaselect.com/146105/article"> Leveraging Structural Diversity and Allosteric Regulatory Mechanisms of Protein Kinases in the Discovery of Small Molecule Inhibitors.</a> <a_journal_title> Curr Med Chem </a_journal_title>. 2016 Oct 6. [Epub ahead of print]<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Leveraging+Structural+Diversity+and+Allosteric+Regulatory+Mechanisms+of+Protein+Kinases+in+the+Discovery+of+Small+Molecule+Inhibitors"> PUBMED.</a><a href="#abstractLeverage" id="btn2"> Abstract.</a><br><br>
     	
		  <a_abstract NAME="abstractExplo" id="x2"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> Protein kinases are versatile molecule switches that govern functional processes in signal transduction networks and regulate fundamental biological processes of cell cycle and organism development. The continuous growth of biological information and a remarkable breath of structural, genetic, and pharmacological studies on protein kinase genes have significantly advanced our knowledge of the kinase activation, drug binding and allosteric mechanisms underlying kinase regulation and interactions in signaling cascades. . Structural and biochemical studies of the genetic and molecular determinants of protein kinases binding with inhibitors have been the cornerstone of drug discovery efforts in clinical oncology leading to proliferation of effective anticancer therapies. Recent advances in understanding allosteric regulation of protein kinases have fueled unprecedented efforts aiming in the discovery of targeted and allosteric kinase inhibitors that can combat cancer mutants and are at the forefront of the precision medicine initiative in oncology. Despite diversity of regulatory scenarios underlying kinase functions, dimerization-driven activation is a common mechanism of allosteric regulation that is shared by many protein kinase families, most notably ErbB and BRAF kinases that play a central role in growth factor signaling and human disease. In this review, we focused on structural, biochemical and computational studies of the ErbB and BRAF kinases and discuss how diversity of the structural landscape for these kinase genes and dimerization-dependent mechanisms of their regulation can be leveraged in the design and discovery of kinase inhibitors and allosteric modulators of kinase activation. The lessons from this analysis could inform discovery of specific targeted therapies and robust drug combinations for cancer treatment. </form> <br><br></a_abstract>
      
		  Tse A, Verkhivker GM. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0166583"> Exploring Molecular Mechanisms of Paradoxical Activation in the BRAF Kinase Dimers: Atomistic Simulations of Conformational Dynamics and Modeling of Allosteric Communication Networks and Signaling Pathways.</a><a_journal_title> Plos ONE </a_journal_title>, 2016, Nov 18;<a_journal_volume>11(11):e0166583.</a_journal_volume><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Exploring+Molecular+Mechanisms+of+Paradoxical+Activation+in+the+BRAF+Kinase+Dimers%3A+Atomistic+Simulations+of+Conformational+Dynamics+and+Modeling+of+Allosteric+Communication+Networks+and+Signaling+Pathways"> PUBMED.</a><a href="#abstractExplo" id="btn3"> Abstract.</a><br><br>
        
		  <a_abstract NAME="abstractExplo" id="x3"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> The recent studies have revealed that most BRAF inhibitors can paradoxically induce kinase activation by promoting dimerization and enzyme transactivation. Despite rapidly growing number of structural and functional studies about the BRAF dimer complexes, the molecular basis of paradoxical activation phenomenon is poorly understood and remains largely hypothetical. In this work, we have explored the relationships between inhibitor binding, protein dynamics and allosteric signaling in the BRAF dimers using a network-centric approach. Using this theoretical framework, we have combined molecular dynamics simulations with coevolutionary analysis and modeling of the residue interaction networks to determine molecular determinants of paradoxical activation. We have investigated functional effects produced by paradox inducer inhibitors PLX4720, Dabrafenib, Vemurafenib and a paradox breaker inhibitor PLX7904. Functional dynamics and binding free energy analyses of the BRAF dimer complexes have suggested that negative cooperativity effect and dimer-promoting potential of the inhibitors could be important drivers of paradoxical activation. We have introduced a protein structure network model in which coevolutionary residue dependencies and dynamic maps of residue correlations are integrated in the construction and analysis of the residue interaction networks. The results have shown that coevolutionary residues in the BRAF structures could assemble into independent structural modules and form a global interaction network that may promote dimerization. We have also found that BRAF inhibitors could modulate centrality and communication propensities of global mediating centers in the residue interaction networks. By simulating allosteric communication pathways in the BRAF structures, we have determined that paradox inducer and breaker inhibitors may activate specific signaling routes that correlate with the extent of paradoxical activation. While paradox inducer inhibitors may facilitate a rapid and efficient communication via an optimal single pathway, the paradox breaker may induce a broader ensemble of suboptimal and less efficient communication routes. The central finding of our study is that paradox breaker PLX7904 could mimic structural, dynamic and network features of the inactive BRAF-WT monomer that may be required for evading paradoxical activation. The results of this study rationalize the existing structure-functional experiments by offering a network-centric rationale of the paradoxical activation phenomenon. We argue that BRAF inhibitors that amplify dynamic features of the inactive BRAF-WT monomer and intervene with the allosteric interaction networks may serve as effective paradox breakers in cellular environment.</form><br><br></a_abstract>
      
      
		  Verkhivker GM. <a href="http://dx.doi.org/10.1039/c6mb00298f"> Molecular dynamics simulations and modelling of the residue interaction networks in the BRAF kinase complexes with small molecule inhibitors: probing the allosteric effects of ligand-induced kinase dimerization and paradoxical activation.</a> <a_journal_title>  Mol Biosyst </a_journal_title>. 2016 Oct 20;<a_journal_volume>12(10):3146-65.</a_journal_volume> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1039%2Fc6mb00298f"> PUBMED.</a> <a href="#abstractMolDyn" id="btn4">Abstract.</a><br><br>
      	
		  <a_abstract NAME="abstractMolDyn" id="x4"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> Protein kinases are central to proper functioning of cellular networks and are an integral part of many signal transduction pathways. The family of protein kinases represents by far the largest and most important class of therapeutic targets in oncology. Dimerization-induced activation has emerged as a common mechanism of allosteric regulation in BRAF kinases, which play an important role in growth factor signalling and human diseases. Recent studies have revealed that most of the BRAF inhibitors can induce dimerization and paradoxically stimulate enzyme transactivation by conferring an active conformation in the second monomer of the kinase dimer. The emerging connections between inhibitor binding and BRAF kinase domain dimerization have suggested a molecular basis of the activation mechanism in which BRAF inhibitors may allosterically modulate the stability of the dimerization interface and affect the organization of residue interaction networks in BRAF kinase dimers. In this work, we integrated structural bioinformatics analysis, molecular dynamics and binding free energy simulations with the protein structure network analysis of the BRAF crystal structures to determine dynamic signatures of BRAF conformations in complexes with different types of inhibitors and probe the mechanisms of the inhibitor-induced dimerization and paradoxical activation. The results of this study highlight previously unexplored relationships between types of BRAF inhibitors, inhibitor-induced changes in the residue interaction networks and allosteric modulation of the kinase activity. This study suggests a mechanism by which BRAF inhibitors could promote or interfere with the paradoxical activation of BRAF kinases, which may be useful in informing discovery efforts to minimize the unanticipated adverse biological consequences of these therapeutic agents.</form><br><br></a_abstract> 
    
    
		  Stetz G, Verkhivker GM. <a href="http://pubs.acs.org/doi/abs/10.1021/acs.jcim.5b00755"> Probing Allosteric Inhibition Mechanisms of the Hsp70 Chaperone Proteins Using Molecular Dynamics Simulations and Analysis of the Residue Interaction Networks.</a> <a_journal_title>J Chem Inf Model.</a_journal_title> 2016 Aug 22;<a_journal_volume>56(8):1490-517</a_journal_volume>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Probing+Allosteric+Inhibition+Mechanisms+of+the+Hsp70+Chaperone+Proteins+Using+Molecular+Dynamics+Simulations+and+Analysis+of+the+Residue+Interaction+Networks"> PUBMED. </a> <a href="#abstractProb" id="btn5">Abstract.</a><br><br>
         
		  <a_abstract NAME="abstractProb" id="x5"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title>  Although molecular mechanisms of allosteric regulation in the Hsp70 chaperones have been extensively studied at both structural and functional levels, the current understanding of allosteric inhibition of chaperone activities by small molecules is still lacking. In the current study, using a battery of computational approaches, we probed allosteric inhibition mechanisms of E. coli Hsp70 (DnaK) and human Hsp70 proteins by small molecule inhibitors PET-16 and novolactone. Molecular dynamics simulations and binding free energy analysis were combined with network-based modeling of residue interactions and allosteric communications to systematically characterize and compare molecular signatures of the apo form, substrate-bound, and inhibitor-bound chaperone complexes. The results suggested a mechanism by which the allosteric inhibitors may leverage binding energy hotspots in the interaction networks to stabilize a specific conformational state and impair the interdomain allosteric control. Using the network-based centrality analysis and community detection, we demonstrated that substrate binding may strengthen the connectivity of local interaction communities, leading to a dense interaction network that can promote an efficient allosteric communication. In contrast, binding of PET-16 to DnaK may induce significant dynamic changes and lead to a fractured interaction network and impaired allosteric communications in the DnaK complex. By using a mechanistic-based analysis of distance fluctuation maps and allosteric propensities of protein residues, we determined that the allosteric network in the PET-16 complex may be small and localized due to the reduced communication and low cooperativity of the substrate binding loops, which may promote the higher rates of substrate dissociation and the decreased substrate affinity. In comparison with the significant effect of PET-16, binding of novolactone to HSPA1A may cause only moderate network changes and preserve allosteric coupling between the allosteric pocket and the substrate binding region. The impact of novolactone on the conformational dynamics and allosteric communications in the HSPA1A complex was comparable to the substrate effect, which is consistent with the experimental evidence that PET-16, but not novolactone binding, can significantly decrease substrate affinity. We argue that the unique dynamic and network signatures of PET-16 and novolactone may be linked with the experimentally observed functional effects of these inhibitors on allosteric regulation and substrate binding. </form><br><br></a_abstract> 
          
    
		  Verkhivker GM.<a href="http://www.worldscientific.com/doi/abs/10.1142/9789814749411_0005">Integrating genetic and structural data on human protein kinome in network-based modeling of kinase sensitivities and resistance to targeted and personalized anticancer drugs.  </a> <a_journal_title>Pac Symp Biocomput.</a_journal_title> 2016;<a_journal_volume> 21:45-56.</a_journal_volume> <a href="https://www.ncbi.nlm.nih.gov/pubmed/26776172"> PUBMED. </a><a href="#abstractIntegrat" id="btn6">Abstract.</a><br><br>
    	
		  <a_abstract NAME="abstractIntegrat" id="x6"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> The human protein kinome presents one of the largest protein families that orchestrate functional processes in complex cellular networks, and when perturbed, can cause various cancers. The abundance and diversity of genetic, structural, and biochemical data underlies the complexity of mechanisms by which targeted and personalized drugs can combat mutational profiles in protein kinases. Coupled with the evolution of system biology approaches, genomic and proteomic technologies are rapidly identifying and charactering novel resistance mechanisms with the goal to inform rationale design of personalized kinase drugs. Integration of experimental and computational approaches can help to bring these data into a unified conceptual framework and develop robust models for predicting the clinical drug resistance. In the current study, we employ a battery of synergistic computational approaches that integrate genetic, evolutionary, biochemical, and structural data to characterize the effect of cancer mutations in protein kinases. We provide a detailed structural classification and analysis of genetic signatures associated with oncogenic mutations. By integrating genetic and structural data, we employ network modeling to dissect mechanisms of kinase drug sensitivities to oncogenic EGFR mutations. Using biophysical simulations and analysis of protein structure networks, we show that conformational-specific drug binding of Lapatinib may elicit resistant mutations in the EGFR kinase that are linked with the ligand-mediated changes in the residue interaction networks and global network properties of key residues that are responsible for structural stability of specific functional states. A strong network dependency on high centrality residues in the conformation-specific Lapatinib-EGFR complex may explain vulnerability of drug binding to a broad spectrum of mutations and the emergence of drug resistance. Our study offers a systems-based perspective on drug design by unravelling complex relationships between robustness of targeted kinase genes and binding specificity of targeted kinase drugs. We discuss how these approaches can exploit advances in chemical biology and network science to develop novel strategies for rationally tailored and robust personalized drug therapies. </form><br><br></a_abstract>
    	
    
    
   <A NAME="2015"></A>
   <p class="sub_ctrXS">2015</p> 

    
		  Stetz G, Verkhivker GM.  <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0143752"> Dancing through Life: Molecular Dynamics Simulations and Network-Centric Modeling of Allosteric Mechanisms in Hsp70 and Hsp110 Chaperone Proteins.</a>  <a_journal_title>PLoS One. </a_journal_title> 2015 Nov 30;<a_journal_volume>10(11):e0143752.</a_journal_volume> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1371%2Fjournal.pone.0143752"> PUBMED. </a><a href="#abstractDance" id="btn7">Abstract.</a><br><br>
      
		  <a_abstract NAME="abstractDance" id="x7"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> Hsp70 and Hsp110 chaperones play an important role in regulating cellular processes that involve protein folding and stabilization, which are essential for the integrity of signaling networks. Although many aspects of allosteric regulatory mechanisms in Hsp70 and Hsp110 chaperones have been extensively studied and significantly advanced in recent experimental studies, the atomistic picture of signal propagation and energetics of dynamics-based communication still remain unresolved. In this work, we have combined molecular dynamics simulations and protein stability analysis of the chaperone structures with the network modeling of residue interaction networks to characterize molecular determinants of allosteric mechanisms. We have shown that allosteric mechanisms of Hsp70 and Hsp110 chaperones may be primarily determined by nucleotide-induced redistribution of local conformational ensembles in the inter-domain regions and the substrate binding domain. Conformational dynamics and energetics of the peptide substrate binding with the Hsp70 structures has been analyzed using free energy calculations, revealing allosteric hotspots that control negative cooperativity between regulatory sites. The results have indicated that cooperative interactions may promote a population-shift mechanism in Hsp70, in which functional residues are organized in a broad and robust allosteric network that can link the nucleotide-binding site and the substrate-binding regions. A smaller allosteric network in Hsp110 structures may elicit an entropy-driven allostery that occurs in the absence of global structural changes. We have found that global mediating residues with high network centrality may be organized in stable local communities that are indispensable for structural stability and efficient allosteric communications. The network-centric analysis of allosteric interactions has also established that centrality of functional residues could correlate with their sensitivity to mutations across diverse chaperone functions. This study reconciles a wide spectrum of structural and functional experiments by demonstrating how integration of molecular simulations and network-centric modeling may explain thermodynamic and mechanistic aspects of allosteric regulation in chaperones. </form><br><br></a_abstract>
       
		  Tse A, Verkhivker GM. <a href="http://pubs.acs.org/doi/10.1021/acs.jcim.5b00240"> Molecular Dynamics Simulations and Structural Network Analysis of c-Abl and c-Src Kinase Core Proteins: Capturing Allosteric Mechanisms and Communication Pathways from Residue Centrality.</a> <a_journal_title>J Chem Inf Model.</a_journal_title> 2015 Aug 24;<a_journal_volume>55(8):1645-62.</a_journal_volume> Epub 2015 Aug 12. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26236953"> PUBMED. </a><a href="#abstractStructNet" id="btn8">Abstract.</a><br><br>
    
		  <a_abstract NAME="abstractStructNet" id="x8"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> The Abl and Src tyrosine kinases play a fundamental regulatory role in orchestrating functional processes in cellular networks and represent an important class of therapeutic targets. Crystallographic studies of these kinases have revealed a similar structural organization of multidomain complexes that confers salient features of their regulatory mechanisms. Molecular characterization of the interaction networks and regulatory residues by which the SH3 and SH2 domains act cooperatively with the catalytic domain to suppress or promote kinase activation presents an active area of structural, biochemical, and computational investigations. In this work, we combine biophysical simulations with computational modeling of the residue interaction networks to characterize allosteric mechanisms of kinase regulation and gain insight into differential sensitivity of c-Abl and c-Src kinases to specific drug binding. Using these approaches, we examine dynamics of cooperative rearrangements in the residue interaction networks and elucidate the structural role of regulatory residues responsible for modulation of kinase activity. We have found that global network parameters such as residue centrality can unambiguously distinguish functional sites that are responsible for mediating allosteric interactions in the regulatory assemblies. This study has revealed mechanistic aspects of allosteric mechanisms and communication pathways by which the SH3 and SH2 domains may exert their regulatory influence on the catalytic domain and kinase activity. We have also found that high centrality residues can be linked to each other to form efficient and robust routes that transmit allosteric signals between spatially separated regulatory regions. The presented results have demonstrated that global features of the residue interaction networks may serve as transparent and robust indicators of kinase regulatory mechanisms and accurately pinpoint key functional residues.</form><br><br></a_abstract>
     
		  Tse A, Verkhivker GM.<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2015/MB/C5MB00246J#!divAbstract"> Small-world networks of residue interactions in the Abl kinase complexes with cancer drugs: topology of allosteric communication pathways can determine drug resistance effects. </a> <a_journal_title>Mol Biosyst.</a_journal_title> 2015 Jul;<a_journal_volume>11(7):2082-95.</a_journal_volume> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25996455"> PUBMED. </a><a href="#abstractSmallWorld" id="btn9">Abstract.</a><br><br>
       
		  <a_abstract NAME="abstractSmallWorld" id="x9"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> The human protein kinases play a fundamental regulatory role in orchestrating functional processes in complex cellular networks. Understanding how conformational equilibrium between functional kinase states can be modulated by ligand binding or mutations is critical for quantifying molecular basis of allosteric regulation and drug resistance. In this work, molecular dynamics simulations of the Abl kinase complexes with cancer drugs (Imatinib and Dasatinib) were combined with structure-based network modeling to characterize dynamics of the residue interaction networks in these systems. The results have demonstrated that structural architecture of kinase complexes can produce a small-world topology of the interaction networks. Our data have indicated that specific Imatinib binding to a small number of highly connected residues could lead to network-bridging effects and allow for efficient allosteric communication, which is mediated by a dominant pathway sensitive to the unphosphorylated Abl state. In contrast, Dasatinib binding to the active kinase form may activate a broader ensemble of allosteric pathways that are less dependent on the phosphorylation status of Abl and provide a better balance between the efficiency and resilience of signaling routes. Our results have unveiled how differences in the residue interaction networks and allosteric communications of the Abl kinase complexes can be directly related to drug resistance effects. This study offers a plausible perspective on how efficiency and robustness of the residue interaction networks and allosteric pathways in kinase structures may be associated with protein responses to drug binding. </form><br><br></a_abstract>
        
		  Tse A, Verkhivker GM. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0130203"> Molecular Determinants Underlying Binding Specificities of the ABL Kinase Inhibitors: Combining Alanine Scanning of Binding Hot Spots with Network Analysis of Residue Interactions and Coevolution.</a> <a_journal_title> PLoS One. </a_journal_title> 2015 Jun 15;<a_journal_volume>10(6):e0130203.</a_journal_volume> eCollection 2015. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26075886"> PUBMED. </a><a href="#abstractMolecDet" id="btn10">Abstract.</a><br><br> 
       
		  <a_abstract NAME="abstractMolecDet" id="x10"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> Quantifying binding specificity and drug resistance of protein kinase inhibitors is of fundamental importance and remains highly challenging due to complex interplay of structural and thermodynamic factors. In this work, molecular simulations and computational alanine scanning are combined with the network-based approaches to characterize molecular determinants underlying binding specificities of the ABL kinase inhibitors. The proposed theoretical framework unveiled a relationship between ligand binding and inhibitor-mediated changes in the residue interaction networks. By using topological parameters, we have described the organization of the residue interaction networks and networks of coevolving residues in the ABL kinase structures. This analysis has shown that functionally critical regulatory residues can simultaneously embody strong coevolutionary signal and high network centrality with a propensity to be energetic hot spots for drug binding. We have found that selective (Nilotinib) and promiscuous (Bosutinib, Dasatinib) kinase inhibitors can use their energetic hot spots to differentially modulate stability of the residue interaction networks, thus inhibiting or promoting conformational equilibrium between inactive and active states. According to our results, Nilotinib binding may induce a significant network-bridging effect and enhance centrality of the hot spot residues that stabilize structural environment favored by the specific kinase form. In contrast, Bosutinib and Dasatinib can incur modest changes in the residue interaction network in which ligand binding is primarily coupled only with the identity of the gate-keeper residue. These factors may promote structural adaptability of the active kinase states in binding with these promiscuous inhibitors. Our results have related ligand-induced changes in the residue interaction networks with drug resistance effects, showing that network robustness may be compromised by targeted mutations of key mediating residues. This study has outlined mechanisms by which inhibitor binding could modulate resilience and efficiency of allosteric interactions in the kinase structures, while preserving structural topology required for catalytic activity and regulation.</form><br><br></a_abstract>
        
        
      <A NAME="2014"></A>
      <p class="sub_ctrXS">2014</p>
      
      
		  James KA, Verkhivker GM. <a href="http://www.ploscompbiol.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1003679&representation=PDF"> Structure-based network analysis of activation mechanisms in the ErbB family of receptor tyrosine kinases: the regulatory spine residues are global mediators of structural stability and allosteric interactions. </a> <a_journal_title>PLoS One.</a_journal_title> 2014 Nov26;<a_journal_volume>9(11):3113488.</a_journal_volume> doi:10.1371/journal.pone.0113488. eCollection 2014. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Computational+Modeling+of+Allosteric+Regulation+in+the+Hsp90+Chaperones%3A+A+Statistical+Ensemble+Analysis+of+Protein+Structure+Networks+and+Allosteric+Communications."> PUBMED. </a><a href="#abstractStructBase" id="btn11"> Abstract. </a><br><br>
     
		  <a_abstract NAME="abstractStructBase" id="x11"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> A fundamental role of the Hsp90 chaperone in regulating functional activity of diverse protein clients is essential for the integrity of signaling networks. In this work we have combined biophysical simulations of the Hsp90 crystal structures with the protein structure network analysis to characterize the statistical ensemble of allosteric interaction networks and communication pathways in the Hsp90 chaperones. We have found that principal structurally stable communities could be preserved during dynamic changes in the conformational ensemble. The dominant contribution of the inter-domain rigidity to the interaction networks has emerged as a common factor responsible for the thermodynamic stability of the active chaperone form during the ATPase cycle. Structural stability analysis using force constant profiling of the inter-residue fluctuation distances has identified a network of conserved structurally rigid residues that could serve as global mediating sites of allosteric communication. Mapping of the conformational landscape with the network centrality parameters has demonstrated that stable communities and mediating residues may act concertedly with the shifts in the conformational equilibrium and could describe the majority of functionally significant chaperone residues. The network analysis has revealed a relationship between structural stability, global centrality and functional significance of hotspot residues involved in chaperone regulation. We have found that allosteric interactions in the Hsp90 chaperone may be mediated by modules of structurally stable residues that display high betweenness in the global interaction network. The results of this study have suggested that allosteric interactions in the Hsp90 chaperone may operate via a mechanism that combines rapid and efficient communication by a single optimal pathway of structurally rigid residues and more robust signal transmission using an ensemble of suboptimal multiple communication routes. This may be a universal requirement encoded in protein structures to balance the inherent tension between resilience and efficiency of the residue interaction networks. </form><br><br></a_abstract>
     
     
		  Verkhivker GM. <a href="http://onlinelibrary.wiley.com/doi/10.1002/ijch.201300143/pdf"> Computational Studies of Allosteric Regulation in the Hsp90 Molecular Chaperone: From Functional Dynamics and Protein Structure Networks to Allosteric Communications and Targeted Anti-Cancer Modulators.</a> <a_journal_title>Israel Journal of Chemistry. </a_journal_title>  Special Issue: Computational Molecular Biophysics <a_journal_volume>Volume 54, Issue 8-9</a_journal_volume>, pages 1052–1064, August 2014.<a href="#abstractCompStudies" id="btn12"> Abstract. </a><br><br>
     
		  <a_abstract NAME="abstractCompStudies" id="x12"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> Computational studies of allosteric interactions have witnessed a recent renaissance fueled by growing inter- est in the modeling of complex molecular assemblies and biological networks. Allosteric interactions of the molecular chaperone Hsp90 with a diverse array of cochaperones and client proteins allow for molecular communication in signal transduction networks. In this review, recent developments in the understanding of allosteric interactions in the context of structural, functional, and computational studies of the Hsp90 chaperone are discussed. A comprehensive analysis of structural and network-based models of protein allostery is provided. Computational and experimental approaches and advances in the understanding of Hsp90 interactions and regulatory mechanisms are reviewed to provide a sys- tematic and critical view of the current progress and most challenging questions in the field. The current status and future prospects for translational research, bridging the basic science of chaperones with the discovery of anti- cancer therapies, are also highlighted. </form> <br><br></a_abstract>
    
		  Blacklock K, Verkhivker GM. <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003679"> Computational modeling of allosteric regulation in the hsp90 chaperones: a statistical ensemble analysis of protein structure networks and allosteric communications. </a> <a_journal_title> PLoS Comput Biol. </a_journal_title> 2014 Jun 12;<a_journal_volume>10(6):e1003679.</a_journal_volume> doi: 10.1371/journal.pcbi.1003679. eCollection 2014 Jun. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24922508"> PUBMED. </a> <a href="#abstractCompModel" id="btn13"> Abstract. </a><br><br> 
      
		  <a_abstract NAME="abstractCompModel" id="x13"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> A fundamental role of the Hsp90 chaperone in regulating functional activity of diverse protein clients is essential for the integrity of signaling networks. In this work we have combined biophysical simulations of the Hsp90 crystal structures with the protein structure network analysis to characterize the statistical ensemble of allosteric interaction networks and communication pathways in the Hsp90 chaperones. We have found that principal structurally stable communities could be preserved during dynamic changes in the conformational ensemble. The dominant contribution of the inter-domain rigidity to the interaction networks has emerged as a common factor responsible for the thermodynamic stability of the active chaperone form during the ATPase cycle. Structural stability analysis using force constant profiling of the inter-residue fluctuation distances has identified a network of conserved structurally rigid residues that could serve as global mediating sites of allosteric communication. Mapping of the conformational landscape with the network centrality parameters has demonstrated that stable communities and mediating residues may act concertedly with the shifts in the conformational equilibrium and could describe the majority of functionally significant chaperone residues. The network analysis has revealed a relationship between structural stability, global centrality and functional significance of hotspot residues involved in chaperone regulation. We have found that allosteric interactions in the Hsp90 chaperone may be mediated by modules of structurally stable residues that display high betweenness in the global interaction network. The results of this study have suggested that allosteric interactions in the Hsp90 chaperone may operate via a mechanism that combines rapid and efficient communication by a single optimal pathway of structurally rigid residues and more robust signal transmission using an ensemble of suboptimal multiple communication routes. This may be a universal requirement encoded in protein structures to balance the inherent tension between resilience and efficiency of the residue interaction networks.</form><br><br></a_abstract>
       
       
		  Dixit A, Verkhivker GM. <a href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0086547&representation=PDF">Structure-functional prediction and analysis of cancer mutation effects in protein kinases.</a> <a_journal_title> Comput Math Methods Med. </a_journal_title> 2014;<a_journal_volume>2014:653487.</a_journal_volume> doi: 10.1155/2014/653487. Epub 2014 Apr 8. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24817905"> PUBMED. </a> <a href="#abstractStructFunc" id="btn14"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractStructFunc" id="x14"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> A central goal of cancer research is to discover and characterize the functional effects of mutated genes that contribute to tumorigenesis. In this study, we provide a detailed structural classification and analysis of functional dynamics for members of protein kinase families that are known to harbor cancer mutations. We also present a systematic computational analysis that combines sequence and structure-based prediction models to characterize the effect of cancer mutations in protein kinases. We focus on the differential effects of activating point mutations that increase protein kinase activity and kinase-inactivating mutations that decrease activity. Mapping of cancer mutations onto the conformational mobility profiles of known crystal structures demonstrated that activating mutations could reduce a steric barrier for the movement from the basal "low" activity state to the "active" state. According to our analysis, the mechanism of activating mutations reflects a combined effect of partial destabilization of the kinase in its inactive state and a concomitant stabilization of its active-like form, which is likely to drive tumorigenesis at some level. Ultimately, the analysis of the evolutionary and structural features of the major cancer-causing mutational hotspot in kinases can also aid in the correlation of kinase mutation effects with clinical outcomes. </form><br><br></a_abstract>
       
		  Blacklock K, Verkhivker GM. <a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0086547&representation=PDF"> Allosteric regulation of the Hsp90 dynamics and stability by client recruiter cochaperones: protein structure network modeling. </a> <a_journal_title> PLoS One. </a_journal_title> 2014 Jan 20;<a_journal_volume>9(1):e86547.</a_journal_volume> doi: 10.1371/journal.pone.0086547. eCollection 2014. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24466147"> PUBMED. </a><a href="#abstractAlloReg" id="btn15"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractAlloReg" id="x15"><form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> The fundamental role of the Hsp90 chaperone in supporting functional activity of diverse protein clients is anchored by specific cochaperones. A family of immune sensing client proteins is delivered to the Hsp90 system with the aid of cochaperones Sgt1 and Rar1 that act cooperatively with Hsp90 to form allosterically regulated dynamic complexes. In this work, functional dynamics and protein structure network modeling are combined to dissect molecular mechanisms of Hsp90 regulation by the client recruiter cochaperones. Dynamic signatures of the Hsp90-cochaperone complexes are manifested in differential modulation of the conformational mobility in the Hsp90 lid motif. Consistent with the experiments, we have determined that targeted reorganization of the lid dynamics is a unifying characteristic of the client recruiter cochaperones. Protein network analysis of the essential conformational space of the Hsp90-cochaperone motions has identified structurally stable interaction communities, interfacial hubs and key mediating residues of allosteric communication pathways that act concertedly with the shifts in conformational equilibrium. The results have shown that client recruiter cochaperones can orchestrate global changes in the dynamics and stability of the interaction networks that could enhance the ATPase activity and assist in the client recruitment. The network analysis has recapitulated a broad range of structural and mutagenesis experiments, particularly clarifying the elusive role of Rar1 as a regulator of the Hsp90 interactions and a stability enhancer of the Hsp90-cochaperone complexes. Small-world organization of the interaction networks in the Hsp90 regulatory complexes gives rise to a strong correspondence between highly connected local interfacial hubs, global mediator residues of allosteric interactions and key functional hot spots of the Hsp90 activity. We have found that cochaperone-induced conformational changes in Hsp90 may be determined by specific interaction networks that can inhibit or promote progression of the ATPase cycle and thus control the recruitment of client proteins.</form><br><br></a_abstract>
      
       <A NAME="2013"></A>
       <p class="sub_ctrXS">2013</p>
       
       
		  Lawless N, Blacklock K, Berrigan E, Verkhivker GM. <a href="http://www.mdpi.com/1424-8247/6/11/1407/pdf"> Structural bioinformatics and protein docking analysis of the molecular chaperone-kinase interactions: towards allosteric inhibition of protein kinases by targeting the hsp90-cdc37 chaperone machinery.</a> <a_journal_title> Pharmaceuticals (Basel). </a_journal_title> 2013 Nov 11;<a_journal_volume>6(11):1407-28.</a_journal_volume> doi: 10.3390/ph6111407. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24287464"> PUBMED. </a><a href="#abstractStructBio" id="btn16"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractStructBio" id="x16"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> A fundamental role of the Hsp90-Cdc37 chaperone system in mediating maturation of protein kinase clients and supporting kinase functional activity is essential for the integrity and viability of signaling pathways involved in cell cycle control and organism development. Despite significant advances in understanding structure and function of molecular chaperones, the molecular mechanisms and guiding principles of kinase recruitment to the chaperone system are lacking quantitative characterization. Structural and thermodynamic characterization of Hsp90-Cdc37 binding with protein kinase clients by modern experimental techniques is highly challenging, owing to a transient nature of chaperone-mediated interactions. In this work, we used experimentally-guided protein docking to probe the allosteric nature of the Hsp90-Cdc37 binding with the cyclin-dependent kinase 4 (Cdk4) kinase clients. The results of docking simulations suggest that the kinase recognition and recruitment to the chaperone system may be primarily determined by Cdc37 targeting of the N-terminal kinase lobe. The interactions of Hsp90 with the C-terminal kinase lobe may provide additional "molecular brakes" that can lock (or unlock) kinase from the system during client loading (release) stages. The results of this study support a central role of the Cdc37 chaperone in recognition and recruitment of the kinase clients. Structural analysis may have useful implications in developing strategies for allosteric inhibition of protein kinases by targeting the Hsp90-Cdc37 chaperone machinery. </form><br><br></a_abstract>
       
		  Blacklock K, Verkhivker GM. <a href="http://www.medscape.com/medline/abstract/24191708"> Experimentally guided structural modeling and dynamics analysis of Hsp90-p53 interactions: allosteric regulation of the Hsp90 chaperone by a client protein.</a> <a_journal_title> J Chem Inf Model. </a_journal_title> 2013 Nov 25;<a_journal_volume>53(11):2962-78.</a_journal_volume> doi: 10.1021/ci400434g. Epub 2013 Nov 12.<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24191708"> PUBMED. </a><a href="#abstractExpGuide" id="btn17"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractExpGuide" id="x17"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> A fundamental role of the Hsp90 chaperone system in mediating maturation of protein clients is essential for the integrity of signaling pathways involved in cell cycle control and organism development. Molecular characterization of Hsp90 interactions with client proteins is fundamental to understanding the activity of many tumor-inducing signaling proteins and presents an active area of structural and biochemical studies. In this work, we have probed mechanistic aspects of allosteric regulation of Hsp90 by client proteins via a detailed computational study of Hsp90 interactions with the tumor suppressor protein p53. Experimentally guided protein docking and molecular dynamics structural refinement have reconstructed the recognition-competent states of the Hsp90-p53 complexes that are consistent with the NMR studies. Protein structure network analysis has identified critical interacting networks and specific residues responsible for structural integrity and stability of the Hsp90-p53 complexes. Coarse-grained modeling was used to characterize the global dynamics of the regulatory complexes and map p53-induced changes in the conformational equilibrium of Hsp90. The variations in the functional dynamics profiles of the Hsp90-p53 complexes are consistent with the NMR studies and could explain differences in the functional role of the alternative binding sites. Despite the overall similarity of the collective movements and the same global interaction footprint, p53 binding at the C-terminal interaction site of Hsp90 may have a more significant impact on the chaperone dynamics, which is consistent with the stronger allosteric effect of these interactions revealed by the experimental studies. The results suggest that p53-induced modulation of the global dynamics and structurally stable interaction networks can target the regulatory hinge regions and facilitate stabilization of the closed Hsp90 dimer that underlies the fundamental stimulatory effect of the p53 client. </form> <br><br></a_abstract>
      
       
		  Blacklock K, Verkhivker GM.<a href="http://www.researchgate.net/profile/Kristin_Blacklock/publication/256102740_Differential_Modulation_of_Functional_Dynamics_and_Allosteric_Interactions_in_the_Hsp90-Cochaperone_Complexes_with_p23_and_Aha1_A_Computational_Study/links/00b495255acc861427000000?ev=pub_ext_doc_dl&origin=publication_detail&inViewer=true"> Differential modulation of functional dynamics and allosteric interactions in the Hsp90-cochaperone complexes with p23 and Aha1: a computational study.</a> <a_journal_title> PLoS One. </a_journal_title> 2013 Aug 19;<a_journal_volume>8(8):e71936.</a_journal_volume> doi: 10.1371/journal.pone.0071936. eCollection 2013. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23977182"> PUBMED. </a><a href="#abstractDiffModul" id="btn18"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractDiffModul" id="x18"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> Allosteric interactions of the molecular chaperone Hsp90 with a large cohort of cochaperones and client proteins allow for molecular communication and event coupling in signal transduction networks. The integration of cochaperones into the Hsp90 system is driven by the regulatory mechanisms that modulate the progression of the ATPase cycle and control the recruitment of the Hsp90 clientele. In this work, we report the results of computational modeling of allosteric regulation in the Hsp90 complexes with the cochaperones p23 and Aha1. By integrating protein docking, biophysical simulations, modeling of allosteric communications, protein structure network analysis and the energy landscape theory we have investigated dynamics and stability of the Hsp90-p23 and Hsp90-Aha1 interactions in direct comparison with the extensive body of structural and functional experiments. The results have revealed that functional dynamics and allosteric interactions of Hsp90 can be selectively modulated by these cochaperones via specific targeting of the regulatory hinge regions that could restrict collective motions and stabilize specific chaperone conformations. The protein structure network parameters have quantified the effects of cochaperones on conformational stability of the Hsp90 complexes and identified dynamically stable communities of residues that can contribute to the strengthening of allosteric interactions. According to our results, p23-mediated changes in the Hsp90 interactions may provide "molecular brakes" that could slow down an efficient transmission of the inter-domain allosteric signals, consistent with the functional role of p23 in partially inhibiting the ATPase cycle. Unlike p23, Aha1-mediated acceleration of the Hsp90-ATPase cycle may be achieved via modulation of the equilibrium motions that facilitate allosteric changes favoring a closed dimerized form of Hsp90. The results of our study have shown that Aha1 and p23 can modulate the Hsp90-ATPase activity and direct the chaperone cycle by exerting the precise control over structural stability, global movements and allosteric communications in Hsp90.</form><br><br></a_abstract>
     
        <A NAME="2012"></A> 
         <p class="sub_ctrXS">2012</p>
         
           
		  Dixit A, Verkhivker GM.<a href="http://www.medscape.com/medline/abstract/22992037"> Integrating ligand-based and protein-centric virtual screening of kinase inhibitors using ensembles of multiple protein kinase genes and conformations. </a> <a_journal_title> J Chem Inf Model. </a_journal_title> 2012 Oct 22;<a_journal_volume>52(10):2501-15.</a_journal_volume> doi: 10.1021/ci3002638. Epub 2012 Oct 1. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22992037"> PUBMED. </a><a href="#abstractIntLig" id="btn19"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractIntLig" id="x19"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> The rapidly growing wealth of structural and functional information about kinase genes and kinase inhibitors that is fueled by a significant therapeutic role of this protein family provides a significant impetus for development of targeted computational screening approaches. In this work, we explore an ensemble-based, protein-centric approach that allows for simultaneous virtual ligand screening against multiple kinase genes and multiple kinase receptor conformations. We systematically analyze and compare the results of ligand-based and protein-centric screening approaches using both single-receptor and ensemble-based docking protocols. A panel of protein kinase targets that includes ABL, EGFR, P38, CDK2, TK, and VEGFR2 kinases is used in this comparative analysis. By applying various performance metrics we have shown that ligand-centric shape matching can provide an effective enrichment of active compounds outperforming single-receptor docking screening. However, ligand-based approaches can be highly sensitive to the choice of inhibitor queries. Employment of multiple inhibitor queries combined with parallel selection ranking criteria can improve the performance and efficiency of ligand-based virtual screening. We also demonstrated that replica-exchange Monte Carlo docking with kinome-based ensembles of multiple crystal structures can provide a superior early enrichment on the kinase targets. The central finding of this study is that incorporation of the template-based structural information about kinase inhibitors and protein kinase structures in diverse functional states can significantly enhance the overall performance and robustness of both ligand and protein-centric screening strategies. The results of this study may be useful in virtual screening of kinase inhibitors potentially offering a beneficial spectrum of therapeutic activities across multiple disease states. </form> <br><br></a_abstract>
     
       
		  Verkhivker GM. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3397965/pdf/pone.0040897.pdf">Simulating Molecular Mechanisms of the MDM2-Mediated Regulatory Interactions: A Conformational Selection Model of the MDM2 Lid Dynamics.</a> <a_journal_title> Plos One </a_journal_title> 2012;<a_journal_volume>7(7):e40897.</a_journal_volume> Epub 2012 Jul 16.<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Simulating+Molecular+Mechanisms+of+the+MDM2-Mediated+Regulatory+Interactions%3A+A+Conformational+Selection+Model+of+the+MDM2+Lid+Dynamics."> PUBMED. </a><a href="#abstractSimulating" id="btn20"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractSimulating" id="x20"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> Diversity and complexity of MDM2 mechanisms govern its principal function as the cellular antagonist of the p53 tumor suppressor. Structural and biophysical studies have demonstrated that MDM2 binding could be regulated by the dynamics of a pseudo-substrate lid motif. However, these experiments and subsequent computational studies have produced conflicting mechanistic models of MDM2 function and dynamics. We propose a unifying conformational selection model that can reconcile experimental findings and reveal a fundamental role of the lid as a dynamic regulator of MDM2-mediated binding. In this work, structure, dynamics and energetics of apo-MDM2 are studied as a function of posttranslational modifications and length of the lid. We found that the dynamic equilibrium between ‘‘closed’’ and ‘‘semi-closed’’ lid forms may be a fundamental characteristic of MDM2 regulatory interactions, which can be modulated by phosphorylation, phosphomimetic mutation as well as by the lid size. Our results revealed that these factors may regulate p53-MDM2 binding by fine-tuning the thermodynamic equilibrium between preexisting conformational states of apo-MDM2. In agreement with NMR studies, the effect of phosphorylation on MDM2 interactions was more pronounced with the truncated lid variant that favored the thermodynamically dominant closed form. The phosphomimetic mutation S17D may alter the lid dynamics by shifting the thermodynamic equilibrium towards the ensemble of ‘‘semi-closed’’ conformations. The dominant ‘‘semi- closed’’ lid form and weakened dependence on the phosphorylation seen in simulations with the complete lid can provide a rationale for binding of small p53-based mimetics and inhibitors without a direct competition with the lid dynamics. The results suggested that a conformational selection model of preexisting MDM2 states may provide a robust theoretical framework for understanding MDM2 dynamics. Probing biological functions and mechanisms of MDM2 regulation would require further integration of computational and experimental studies and may help to guide drug design of novel anti- cancer therapeutics. </form> <br><br></a_abstract>
        
        
		  Dixit A, Verkhivker GM.<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3356286/pdf/pone.0037605.pdf"> Probing molecular mechanisms of the hsp90 chaperone: biophysical modeling identifies key regulators of functional dynamics.</a> <a_journal_title>PLoS One. </a_journal_title> 2012;<a_journal_volume>7(5):e37605.</a_journal_volume> Epub 2012 May 18.<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Probing+Molecular+Mechanisms+of+the+Hsp90+Chaperone%3A+Biophysical+Modeling+Identifies+Key+Regulators+of+Functional+Dynamics">  PUBMED. </a><a href="#abstractProbing" id="btn21"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractProbing" id="x21"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> Deciphering functional mechanisms of the Hsp90 chaperone machinery is an important objective in cancer biology aiming to facilitate discovery of targeted anti-cancer therapies. Despite significant advances in understanding structure and function of molecular chaperones, organizing molecular principles that control the relationship between conformational diversity and functional mechanisms of the Hsp90 activity lack a sufficient quantitative characterization. We combined molecular dynamics simulations, principal component analysis, the energy landscape model and structure-functional analysis of Hsp90 regulatory interactions to systematically investigate functional dynamics of the molecular chaperone. This approach has identified a network of conserved regions common to the Hsp90 chaperones that could play a universal role in coordinating functional dynamics, principal collective motions and allosteric signaling of Hsp90. We have found that these functional motifs may be utilized by the molecular chaperone machinery to act collectively as central regulators of Hsp90 dynamics and activity, including the inter-domain communications, control of ATP hydrolysis, and protein client binding. These findings have provided support to a long-standing assertion that allosteric regulation and catalysis may have emerged via common evolutionary routes. The interaction networks regulating functional motions of Hsp90 may be determined by the inherent structural architecture of the molecular chaperone. At the same time, the thermodynamics- based ‘‘conformational selection’’ of functional states is likely to be activated based on the nature of the binding partner. This mechanistic model of Hsp90 dynamics and function is consistent with the notion that allosteric networks orchestrating cooperative protein motions can be formed by evolutionary conserved and sparsely connected residue clusters. Hence, allosteric signaling through a small network of distantly connected residue clusters may be a rather general functional requirement encoded across molecular chaperones. The obtained insights may be useful in guiding discovery of allosteric Hsp90 inhibitors targeting protein interfaces with co-chaperones and protein binding clients. </form><br><br></a_abstract>
      
      <A NAME="2011"></A> 
      <p class="sub_ctrXS">2010 - 2011</p>
      
      
      
		  Dixit A, Verkhivker GM.<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3188581/pdf/pone.0026071.pdf">The energy landscape analysis of cancer mutations in protein kinases.</a> <a_journal_title>PLoS One.</a_journal_title> 2011;<a_journal_volume>6(10):e26071.</a_journal_volume> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=The+Energy+Landscape+Analysis+of+Cancer+Mutations+in+Protein+Kinases">PUBMED. </a><a href="#abstractEnergy" id="btn22"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractEnergy" id="x22"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> The growing interest in quantifying the molecular basis of protein kinase activation and allosteric regulation by cancer mutations has fueled computational studies of allosteric signaling in protein kinases. In the present study, we combined computer simulations and the energy landscape analysis of protein kinases to characterize the interplay between oncogenic mutations and locally frustrated sites as important catalysts of allostetric kinase activation. While structurally rigid kinase core constitutes a minimally frustrated hub of the catalytic domain, locally frustrated residue clusters, whose interaction networks are not energetically optimized, are prone to dynamic modulation and could enable allosteric conformational transitions. The results of this study have shown that the energy landscape effect of oncogenic mutations may be allosteric eliciting global changes in the spatial distribution of highly frustrated residues. We have found that mutation-induced allosteric signaling may involve a dynamic coupling between structurally rigid (minimally frustrated) and plastic (locally frustrated) clusters of residues. The presented study has demonstrated that activation cancer mutations may affect the thermodynamic equilibrium between kinase states by allosterically altering the distribution of locally frustrated sites and increasing the local frustration in the inactive form, while eliminating locally frustrated sites and restoring structural rigidity of the active form. The energy landsape analysis of protein kinases and the proposed role of locally frustrated sites in activation mechanisms may have useful implications for bioinformatics-based screening and detection of functional sites critical for allosteric regulation in complex biomolecular systems. </form><br><br></a_abstract>
     
		  Dixit A, Verkhivker GM.<a href="http://www.ploscompbiol.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002179&representation=PDF"> Computational Modeling of Allosteric Communication Reveals Organizing Principles of Mutation-Induced Signaling in ABL and EGFR Kinases.</a> <a_journal_title> PLoS Comput Biol. </a_journal_title> 2011 Oct;<a_journal_volume>7(10):e1002179.</a_journal_volume> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Computational+Modeling+of+Allosteric+Communication+Reveals+Organizing+Principles+of+Mutation-Induced+Signaling+in+ABL+and+EGFR+Kinases.">PUBMED. </a><a href="#abstractModelAllo" id="btn23"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractModelAllo" id="x23"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> The emerging structural information about allosteric kinase complexes and the growing number of allosteric inhibitors call for a systematic strategy to delineate and classify mechanisms of allosteric regulation and long-range communication that control kinase activity. In this work, we have investigated mechanistic aspects of long-range communications in ABL and EGFR kinases based on the results of multiscale simulations of regulatory complexes and computational modeling of signal propagation in proteins. These approaches have been systematically employed to elucidate organizing molecular principles of allosteric signaling in the ABL and EGFR multi-domain regulatory complexes and analyze allosteric signatures of the gate- keeper cancer mutations. We have presented evidence that mechanisms of allosteric activation may have universally evolved in the ABL and EGFR regulatory complexes as a product of a functional cross-talk between the organizing aF-helix and conformationally adaptive aI-helix and aC-helix. These structural elements form a dynamic network of efficiently communicated clusters that may control the long-range interdomain coupling and allosteric activation. The results of this study have unveiled a unifying effect of the gate-keeper cancer mutations as catalysts of kinase activation, leading to the enhanced long-range communication among allosterically coupled segments and stabilization of the active kinase form. The results of this study can reconcile recent experimental studies of allosteric inhibition and long-range cooperativity between binding sites in protein kinases. The presented study offers a novel molecular insight into mechanistic aspects of allosteric kinase signaling and provides a quantitative picture of activation mechanisms in protein kinases at the atomic level. </form><br><br></a_abstract>
      
      
		  Matts RL, Dixit A, Peterson LB, Sun L, Voruganti S, Kalyanaraman, Hartson SD, Verkhivker GM, Blagg BSJ.<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3164513/pdf/nihms-295676.pdf"> Elucidation and assessment of the Hsp90 C-terminal inhibitor binding site. </a> <a_journal_title>ACS Chem Biol.</a_journal_title> 2011 Aug 19;<a_journal_volume>6(8):800-807.</a_journal_volume> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21548602">PUBMED. </a><a href="#abstractElucid" id="btn24"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractElucid" id="x24"> <form style="border:1px solid;height: 100%;"> <a_abstract_title> ABSTRACT: </a_abstract_title> The Hsp90 chaperone machine is required for the folding, activation, and/or stabilization of more than 50 proteins directly related to malignant progression. Hsp90 contains small molecule binding sites at both its N- and C-terminal domains; however, limited structural and biochemical data regarding the C-terminal binding site is available. In this report, the small molecule binding site in the Hsp90 C-terminal domain was revealed by protease fingerprinting and photoaffinity labeling utilizing LC-MS/MS. The identified site was characterized by generation of a homology model for hHsp90α using the SAXS open structure of HtpG and docking the bioactive conformation of NB into the generated model. The resulting model for the bioactive conformation of NB bound to Hsp90α is presented herein. </form><br><br></a_abstract>
     
     
		  Matts RL, Brandt GEL, Lu Y, Dixit A, Mollapour M, Wang S, Donnelly AC, Neckers L, Verkhivker GM, Blagg BSJ. <a href="http://www.sciencedirect.com/science/article/pii/S0968089610009521">A systematic protocol for the characterization of Hsp90 modulators.</a> <a_journal_title> Bioorg Med Chem.</a_journal_title> 2011 <a_journal_volume>19(1):684-92.</a_journal_volume> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=A+systematic+protocol+for+the+characterization+of+Hsp90+modulators.">PUBMED. </a><a href="#abstractProtocol" id="btn25"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractProtocol" id="x25"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> Several Hsp90 modulators have been identified including the N-terminal ligand geldanamycin (GDA), the C-terminal ligand novobiocin (NB), and the co-chaperone disruptor celastrol. Other Hsp90 modulators elicit a mechanism of action that remains unknown. For example, the natural product gedunin and the synthetic anti-spermatogenic agent H2-gamendazole, recently identified Hsp90 modulators, manifest biological activity through undefined mechanisms. Herein, we report a series of biochemical techniques used to classify such modulators into identifiable categories. Such studies provided evidence that gedunin and H2-gamendazole both modulate Hsp90 via a mechanism similar to celastrol, and unlike NB or GDA.</form><br><br></a_abstract>
     
     
	  Friedman A, Torkamani A, Verkhivker GM, Schork NJ. From coding variants to structure and function insights. In the book <a_journal_title>“Protein Structure” </a_journal_title> Ed.Lauren M. Haggerty  Series:   Protein Science and Engineering Nova Science Publishers, 2011.<a href="#abstractCoding" id="btn26"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractCoding" id="x26"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> The availability of cost-effective DNA sequencing technologies has led to the identification and cataloguing of millions of naturally occurring inherited and somatic human genomic variations. As a result, questions concerning the ultimate phenotypic and functional significance of these variations have been raised. Although some large-scale initiatives have been launched for this purpose, including the Encyclopedia of DNA Elements (ENCODE) initiative, there is a need for the DNA sequencing and genomics communities to reach out and foster greater collaborative opportunities with the functional genomics community. One area that is ripe for this kind of activity is the functional characterization of coding variations, as structural proteomics and crystallography researchers may both benefit from, and contribute to, an understanding of the molecular and phenotypic influence of such variations. We briefly review the motivation for this kind of interaction and draw on publicly available data to showcase its need. We also consider how relevant research could be pursued. </form><br><br></a_abstract>
      
		  Morra G, Neves MAC, Plescia CJ, Tsustsumi C, Neckers L, Verkhivker GM, Altieri DC, Colombo G. <a href="http://pubs.acs.org/doi/abs/10.1021/ct100334n"> Dynamics-Based Discovery of Allosteric Inhibitors: Selection of New Ligands for the C-terminal Domain of Hsp90.</a> <a_journal_title>J. Chem. Theory Comput. </a_journal_title>, <a_journal_volume>6 : 2978–2989</a_journal_volume>, 2010. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Dynamics-Based+Discovery+of+Allosteric+Inhibitors%3A+Selection+of+New+Ligands+for+the+C-terminal+Domain+of+Hsp90"> PUBMED. </a> <a href="#abstractDynamicDisc" id="btn27"> Abstract. </a><br><br> 
       
		  <a_abstract NAME="abstractDynamicDisc" id="x27"> <form style="border:1px solid;height: 100%;"><a_abstract_title> ABSTRACT: </a_abstract_title> The study of allosteric functional modulation in dynamic proteins is attracting increasing attention. In particular, the discovery of new allosteric sites may generate novel opportunities and strategies for drug development, overcoming the limits of classical active-site oriented drug design. In this paper, we report on the results of a novel, ab initio, fully computational approach for the discovery of allosteric inhibitors based on the physical characterization of signal propagation mechanisms in proteins and apply it to the important molecular chaperone Hsp90. We first characterize the allosteric “hot spots” involved in interdomain communication pathways from the nucleotide-binding site in the N-domain to the distal C-domain. On this basis, we develop dynamic pharmacophore models to screen drug libraries in the search for small molecules with the functional and conformational properties necessary to bind these “hot spot” allosteric sites. Experimental tests show that the selected moelcules bind the Hsp90 C-domain, exhibit antiproliferative activity in different tumor cell lines, while not affecting proliferation of normal human cells, destabilize Hsp90 client proteins, and disrupt association with several cochaperones known to bind the N- and M-domains of Hsp90. These results prove that the hits alter Hsp90 function by affecting its conformational dynamics and recognition properties through an allosteric mechanism. These findings provide us with new insights on the discovery and development of new allosteric inhibitors that are active on important cellular pathways through computational biology. Though based on the specific case of Hsp90, our approach is general and can readily be extended to other target proteins and pathways.</form><br><br></a_abstract>
      
    <A NAME="B2010"></A> 
    <p class="sub_ctrXS">Before 2010 </p>
     
     <a href="/Publications/2005o2009.php">2005 - 2009 (26 Publications) </a><br>
     <a href="/Publications/2000o2004.php">2000 - 2004 (17 Publications) </a><br>
     <a href="/Publications/1995o1999.php">1995 - 1999 (21 Publications) </a><br>
     <a href="/Publications/1990o1994.php">1990 - 1994 (9 Publications)</a><br>
     <a href="/Publications/o1990.php">Before 1990     (15 Publications) </a><br><br>
      
      
      <p>&nbsp;</p>
      
      
      <!-- InstanceEndEditable -->
      <div class="clear"></div>
    
    
    
    </div>
  </div>
  <footer >
    <div id="bottom">
      <div class="clear"></div>
    </div>
  </footer> <div id="subfooter">©2016 Chapman University • One University Drive, Orange, CA 92866 • Phone: (714) 997-6815</div>
</div>
</body>
<!-- InstanceEnd --></html>