<!DOCTYPE html>
<html lang="en"><!-- InstanceBegin template="/Templates/subpage.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<!-- InstanceBeginEditable name="meta" -->
<!-- InstanceEndEditable -->
<!--[if gte IE 9]><meta http-equiv="X-UA-Compatible" content="IE=Edge"><![endif]-->
<!-- InstanceBeginEditable name="doctitle" -->
<title>Chapman University</title>
<!-- InstanceEndEditable -->
<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" type="text/css" href="../css/harv.css">
<link rel="stylesheet" type="text/css" href="../css/harvdrop.css">
<!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><script src="/js/html5shiv.js"></script><![endif]-->
<script src='http://code.jquery.com/jquery-latest.min.js' type='text/javascript'></script><script src="/js/jquery.ui.totop.js"></script>
<!-- InstanceBeginEditable name="head" -->
<!-- InstanceEndEditable -->
</head>
<body>
<div id="container">
  <div id="smNavBk"><a href="http://compbiosciences.chapman.edu/index.php"><img src="../images/header400light.png" width="332" height="58" alt="Chapman University" style="float: left; margin-left: 10px; margin-top: 10px;"></a>
    <div id="collegeLinks" style="font-size: 24px;">
      <p><a href="https://www.chapman.edu/scst/"><strong><img src="../images/xdz1wyqhwtmxxekidsyy.jpg" width="35" height="35" alt=""/>Schmid College of Science and Technology</strong></a></p>
      <p><a href="http://www.chapman.edu/"><img src="../images/open-graph-cu-logo.png" alt="Chapman University" width="35" height="30" title="Chapman University"/> <span style="font-style: normal; font-size: 18pt;">Chapman University</span></a></p>
    </div>
    <div class="clear"></div>
  </div>
  </header>
  <nav id="topMenu" class="desktop"><!-- #BeginLibraryItem "/Library/topMenu.lbi" --> <nav id="topMenu" class="desktop"><ul id="TopNav" class="MenuBarHorizontal">
      <li> <a href="../index.php">Home</a></li>
      <li><a href="../members.php">Members</a>          
  <li><a href="research.php" class="MenuBarItemSubmenu">Research</a>
        <ul>
          <li><a href="computationalCancerBiology.php" >Computational Biology and Translational Bioinformatics in Cancer Research</a></li>
          <li><a href="proteinKinasesResearch.php" >Protein Kinases</a></li>
          <li><a href="molecularChaperonesAndCancerResearch.php">Computatoinal Studies of Molecular Chaperones</a></li>
          <li><a href="translationalBioinformatics.php" >Translational Bioinformatics and Drug Discovery</a></li>
          <li><a href="relatedResearch.php">Related Links</a></li>
        </ul>
  </li>
      <li><a href="../Publications/publications.php" class="MenuBarItemSubmenu">Publications</a>          
        <ul>
          <li><a href="../Publications/2015o2019.php">2015-2019</a></li>
          <li><a href="../Publications/2010o2014.php">2010-2014</a></li>
          <li><a href="../Publications/2005o2009.php">2005-2009</a></li>
          <li><a href="../Publications/2000o2004.php">2000-2004</a></li>
          <li><a href="../Publications/1995o1999.php">1995-1999</a></li>
          <li><a href="../Publications/1990o1994.php">1990-1994</a></li>
          <li><a href="../Publications/o1990.php">Before 1990</a></li>
        </ul>
    <li><a href="../aboutUs.php">About Us</a>  </li>
    <li><a href="../contact.php">Contact Us</a></li>
    </ul>
    <div class="clear"></div></nav><!-- #EndLibraryItem --></nav></div>
  <div id="middle">
    <div id="mainsub"><!-- InstanceBeginEditable name="crumbs" --><script>breadcrumb()</script><!-- InstanceEndEditable -->
     
     
     
    <div class="clear"></div>
    
    
   <!-- InstanceBeginEditable name="main_content_center" -->
   <p class="header">Translational Bioinformatics and Drug Discovery</p>
   <p>The greatest challenge of the postgenomic era is to understand the function of genes   and gene products in multiple organisms --including humans– both from fundamental and applied perspectives. This will ultimately enable the design of new diagnostic tools and pharmacological agents and facilitate efficacious treatment of many pathological processes. These developments are largely guided and enabled by recent advances in translational bioinformatics, computer sciences, quantitative biology and system-based medicine focused on understanding of the   biological principles underpinning the heterogeneity of human disease and   facilitating predictive and personalized medicine solutions.  In the course of next decade, health care and medicine will be transformed from reactively treating illness to proactively maintaining health — from an averaged picture of disease to a systems biology view of individualized care. The complexity of human disease and cancer   requires an integrated approach using computational and experimental high-throughput genetic, molecular, and clinical phenotype analysis.  We are  focused on the development and application of  an integrated  platform  of computational and experimental approaches to(a) advance identification, prediction and functional analysis of genetic and   molecular  signatures associated with cancer   (b) enable the design of  personalized and system-based cancer medicine to combat  specific genomic profiles;  and (c)  facilitate    biomedical  and clinical  research   on the “bench to bedside” path.    We also expanded our recent efforts in supporting a new pathway-based and network-centric paradigm for quantitative analysis of human disease and in silico discovery of anti-cancer therapeutic agents that is based on targeted polypharmacology of signal transduction networks.</p>
   <p> </p>
   <p>A central goal of cancer research involves the discovery and functional characterization of the mutated genes that are implicated in tumorigenesis. Although the application of DNA sequencing and gene expression microarray technologies has accelerated insights into the molecular basis of cancer, the identification and characterization of genetic variants that influence particular human phenotypes and, most emphatically, susceptibility to common diseases of contemporary public health concern such as diabetes, cancers, and neuropsychiatric illness, have been extraordinarily difficult. Recent studies investigating the genetic determinants of cancer suggest that only some of the genetic alterations contributing to tumorigenesis may be inherited, while somatically acquired mutations can contribute decisively during the transition of a normal cell to a cancer cell.  A systematic understanding of the genetic and molecular determinants of cancers has already begun to have a transformative effect on the study and treatment of cancer, most notably due to discovered patterns of somatically acquired mutations in the protein genes which are commonly associated with the disease. The recent developments in sequencing and functional studies have facilitateda series of integrated studies,   combining genetic and  functional approaches   to identify underlying molecular signatures of   cancer mutations in protein kinase genes.  One of the  focuses of my research program is the development of novel  bioinformaticsand computational biology approaches to systematically and comprehensively study the influence of naturally occurring sequence variation, somatic mutations and drug resistant mutations on protein kinase and molecular chaperone function tounderstand molecular basis of cancer and advance drug discovery of personalized anti-cancer  therapeutics.The protein kinase family is an ideal family to achieve this objective because of the growing wealth of structural and functional information about these genes, as well as the prominent role that protein kinases play as therapeutic targets for cancer intervention.</p>
   <p> </p>
   <p>We  are involved in cross-disciplinary and collaborative  research that combines computational  and systems biology approaches with  chemical genomics and molecular profiling  technologies in   predicting molecular signatures of human disease and applications in discovery of personalized anti-cancer therapeutics. For a number of years,   We have been developing algorithms and computational methods that will facilitate the characterization of the influence of genes and genetic variants on molecular phenotypes. These include the analysis and mining of single nucleotide polymorphisms data, quantitative characterization of the significance of naturally-occurring amino acid substitutions on protein structure, the evaluation of in silico effect of amino acid substitutions on protein function, protein-protein and protein-ligand interactions, and the development of algorithms and tools for integration of in silico predictions with in vitro/in vivo functional studies.  Our research focuses on the biological significance of genetic variations identified in tumors, and is based on integration of  genetic, functional, and structural  insights  into the molecular basis of   tumorigenesis.   We have been employing multidisciplinary approaches and collaborations with Scripps  Genomic Medicine and Bioinformatics Core of Scripps Translational Science Institute to develop a pipeline of  bioinformatics analyses and structurally informed functionalannotations and predictions of somatic alterations in the kinase genes based on  targeted resequencing of breast cancer tumors.We are developing  a systematic platform of computational approaches, models and tools to facilitate genome-wide identification, prediction and functional characterization of molecular signatures of cancer-causing mutations. We are embarked on a comprehensive in silico functional profiling  of candidatemutations  identified in genome wide screens to determine which mutations contribute to transformation andrepresent true therapeutic  targets for the treatment of humanmalignancies. We  integrate  machine learning and Bayesian feature selection methods  with pathway-based genetic association analysis, network reconstruction,  and structure-based analysis  to identify  functionally related gene modules  affected by somatic mutations,  The  bioinformatics pipeline of computational approaches  is then integrated  with experimental tools to allow a quantitative functional analysis of SNPs and cancer mutations  and decipher  how variations in protein kinase sequence, evolution  and structure can lead to complex disease phenotypes.  The ultimate overarching goal is to understand genetic and molecular mechanisms of  human disease andintegrate   predictive cancer biomarkers into computational and experimental chemical genomics strategies to identify and design chemical probes of clinical significance. In collaboration with Scripps Translational Science Institute, we are developing a prototype of a  “bench to bedside” path : From  kinome-wide resequencing  and functional  analysis of mutants to  predictive disease biomarkers, drug targets and discovery  of personalized medicine targeting  genomic variants in cancer.</p>
   <p>&nbsp;</p>
   <p>The developed prototype of a  “bench to bedside” path for kinome-wide analysis of cancer mutations and design of targeted therapies.</p>
   <p>&nbsp;</p>
   <p class="sub_ctrXS"> MOLECULAR CHAPERONES AND DRUG  DISCOVERY</p>
   <p>The molecular chaperone Hsp90 (90 kDa heat-shock protein) is a remarkably versatile protein that mediates several fundamental cellular pathwaysinvolved in the cell proliferation, cell survival, and cellular stress response.  Hsp90 is a fundamental hub in protein interaction networks and is critically involved in the hallmark traits of malignancy.  Our research is focused on the development and integration of computational and experimental approaches with the goal to advance understanding of the molecular chaperone mechanisms at atomic resolution and facilitate discovery of novel anti-cancer therapies. These approaches allowed  to  develop and validate  mathematical and  structural models of Hsp90 allosteric regulation  and signal communication pathways. Client proteins of Hsp90 include protein kinases, transcription factors, and other proteins that serve as nodal points in integrating cellular responses to multiple signals. Deregulation of pathways involving these proteins is commonly associated with cancer pathologies. By disabling multiple signaling circuitries, Hsp90 inhibition provides a novel and powerful therapeutic strategy in cancer research, selective for specific cancer mechanisms, yet broadly applicable to disparate tumors with different genetic signatures. Hsp90 Inhibition can suppresses signaling of kinase cancer  mutant clients and overcomes drug resistance. Computational profiling and  systems-based approaches are integrated with experimental strategies  into a systematic platform for  selective targeting of  Hsp90-kinase protein networks. The developed allosteric Hsp90 modulators  can function as specific and personalized therapeutics  for inhibiting protein kinase clients and cancer mutants. We have established a close partnership and are involved in collaborative efforts with a number of prominent Hsp90 research groups including Dr. Matts (Oklahoma State University), Dr. Neckers (National Cancer Institute), Dr. Altieri (University of Massachusetts Medical School), Dr. Agard (UCSF), Dr. Pearl (The Institute of Cancer Research, UK) and others. The expertise   of our research team   and cross-disciplinary collaborations with the Hsp90 experts in molecular and cell biology, biochemistry, NMR, and structural biology will help to develop and validate computational approaches, refine and enhance experimental tools to advance our understanding of the molecular chaperone mechanisms.The insights about  molecular mechanisms and function of molecular chaperone are employed  in chemical genomics-based  profiling, design and biological validation of novel therapeutics of signal transduction networks.</p>
   <p> </p>
   <p>Integrative pipeline of computational and experimental  approaches  for discovery of allosteric Hsp90 modulators as personalized anti-cancer  therapeutics</p>
   <!-- InstanceEndEditable -->
      <div class="clear"></div>
    
    
    
    </div>
  </div>
  <footer >
    <div id="bottom">
      <div class="clear"></div>
    </div>
  </footer> <div id="subfooter">©2016 Chapman University • One University Drive, Orange, CA 92866 • Phone: (714) 997-6815</div>
</div>
</body>
<!-- InstanceEnd --></html>